# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR Red Hat, Inc.
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: anaconda 15.28\n"
"Report-Msgid-Bugs-To: anaconda-devel-list@redhat.com\n"
"POT-Creation-Date: 2011-04-26 10:18-0400\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"

#: anaconda:322
msgid "Press <enter> for a shell"
msgstr ""

#: anaconda:336
#, c-format
msgid ""
"%s requires %s MB of memory to install, but you only have %s MB on this "
"machine.\n"
msgstr ""

#: anaconda:338
#, c-format
msgid ""
"%s requires %s MB of memory to install using this installation method, but "
"you only have %s MB on this machine.\n"
msgstr ""

#: anaconda:341
#, c-format
msgid ""
"The %s graphical installer requires %s MB of memory, but you only have %s MB."
msgstr ""

#: anaconda:346
msgid "Not enough RAM"
msgstr ""

#: anaconda:347
msgid ""
" Try the text mode installer by running:\n"
"\n"
"'/usr/bin/liveinst -T'\n"
"\n"
" from a root terminal."
msgstr ""

#: anaconda:350
msgid " Starting text mode."
msgstr ""

#: anaconda:374 pyanaconda/rescue.py:346 pyanaconda/rescue.py:378
#: pyanaconda/rescue.py:391 pyanaconda/rescue.py:475 pyanaconda/rescue.py:492
#: pyanaconda/rescue.py:503 pyanaconda/text.py:565 loader/windows.h:30
#: loader/cdinstall.c:186 loader/cdinstall.c:190 loader/cdinstall.c:210
#: loader/cdinstall.c:213 loader/cdinstall.c:337 loader/cdinstall.c:340
#: loader/cdinstall.c:416 loader/dirbrowser.c:137 loader/driverdisk.c:208
#: loader/driverdisk.c:392 loader/driverdisk.c:424 loader/driverdisk.c:455
#: loader/driverdisk.c:490 loader/driverdisk.c:506 loader/driverdisk.c:518
#: loader/driverdisk.c:526 loader/driverdisk.c:698 loader/driverselect.c:79
#: loader/driverselect.c:171 loader/hdinstall.c:222 loader/hdinstall.c:282
#: loader/hdinstall.c:300 loader/hdinstall.c:309 loader/hdinstall.c:344
#: loader/hdinstall.c:383 loader/hdinstall.c:398 loader/kbd.c:118
#: loader/kickstart.c:176 loader/kickstart.c:308 loader/kickstart.c:373
#: loader/kickstart.c:922 loader/lang.c:113 loader/lang.c:362
#: loader/loader.c:422 loader/loader.c:458 loader/loader.c:498
#: loader/loader.c:516 loader/loader.c:533 loader/loader.c:570
#: loader/loader.c:1216 loader/loader.c:1244 loader/loader.c:1389
#: loader/mediacheck.c:47 loader/mediacheck.c:86 loader/mediacheck.c:93
#: loader/mediacheck.c:97 loader/mediacheck.c:106 loader/method.c:194
#: loader/method.c:268 loader/net.c:624 loader/net.c:1004 loader/net.c:1813
#: loader/net.c:1832 loader/net.c:1844 loader/nfsinstall.c:93
#: loader/nfsinstall.c:255 loader/urlinstall.c:139 loader/urlinstall.c:161
#: loader/urlinstall.c:213 loader/urls.c:264 loader/urls.c:335
#: loader/urls.c:343 loader/urls.c:350 pyanaconda/textw/constants_text.py:44
msgid "OK"
msgstr ""

#: anaconda:758
msgid "Would you like to use VNC?"
msgstr ""

#: anaconda:759
msgid ""
"Text mode provides a limited set of installation options.  It does not allow "
"you to specify your own partitioning layout or package selections.  Would "
"you like to use VNC mode instead?"
msgstr ""

#: anaconda:785
msgid "Graphical installation is not available. Starting text mode."
msgstr ""

#: anaconda:793
msgid "DISPLAY variable not set. Starting text mode."
msgstr ""

#: pyanaconda/backend.py:132
#, python-format
msgid "Upgrading %s\n"
msgstr ""

#: pyanaconda/backend.py:134
#, python-format
msgid "Installing %s\n"
msgstr ""

#: pyanaconda/bootloader.py:55 pyanaconda/bootloader.py:221
#: pyanaconda/bootloader.py:227 pyanaconda/gui.py:1148 pyanaconda/image.py:73
#: pyanaconda/installinterfacebase.py:51
#: pyanaconda/installinterfacebase.py:130 pyanaconda/packages.py:320
#: pyanaconda/text.py:440 pyanaconda/yuminstall.py:1498
#: pyanaconda/yuminstall.py:1623 pyanaconda/yuminstall.py:1658
#: pyanaconda/iw/blpasswidget.py:149 pyanaconda/iw/upgrade_swap_gui.py:190
#: pyanaconda/iw/upgrade_swap_gui.py:198 pyanaconda/iw/upgrade_swap_gui.py:205
#: pyanaconda/textw/upgrade_text.py:193
msgid "Warning"
msgstr ""

#: pyanaconda/bootloader.py:56
msgid ""
"Filesystems have already been activated.  You cannot go back past this "
"point.\n"
"\n"
"Would you like to continue with the installation?"
msgstr ""

#: pyanaconda/bootloader.py:60 pyanaconda/gui.py:1171 pyanaconda/gui.py:1208
#: pyanaconda/gui.py:1410 pyanaconda/image.py:82 pyanaconda/livecd.py:167
#: pyanaconda/packages.py:137 pyanaconda/upgrade.py:58
#: pyanaconda/upgrade.py:98 pyanaconda/upgrade.py:251
#: pyanaconda/yuminstall.py:259 pyanaconda/yuminstall.py:824
#: pyanaconda/yuminstall.py:1002 pyanaconda/yuminstall.py:1007
#: pyanaconda/yuminstall.py:1086 pyanaconda/yuminstall.py:1092
#: pyanaconda/yuminstall.py:1246 pyanaconda/yuminstall.py:1261
#: pyanaconda/yuminstall.py:1316 pyanaconda/yuminstall.py:1507
#: pyanaconda/yuminstall.py:1526 pyanaconda/yuminstall.py:1549
#: pyanaconda/storage/__init__.py:113 pyanaconda/storage/__init__.py:122
#: pyanaconda/storage/__init__.py:251 pyanaconda/storage/__init__.py:2034
msgid "_Exit installer"
msgstr ""

#: pyanaconda/bootloader.py:60 pyanaconda/image.py:83
#: pyanaconda/kickstart.py:1424 pyanaconda/kickstart.py:1463
#: pyanaconda/upgrade.py:58 pyanaconda/upgrade.py:98
#: pyanaconda/yuminstall.py:1321 pyanaconda/yuminstall.py:1508
#: pyanaconda/iw/partition_gui.py:1602 pyanaconda/storage/__init__.py:2035
msgid "_Continue"
msgstr ""

#: pyanaconda/bootloader.py:158
msgid "Bootloader"
msgstr ""

#: pyanaconda/bootloader.py:158
msgid "Installing bootloader."
msgstr ""

#: pyanaconda/bootloader.py:222
msgid ""
"There was an error installing the bootloader.  The system may not be "
"bootable."
msgstr ""

#: pyanaconda/bootloader.py:228
msgid ""
"No kernel packages were installed on the system.  Bootloader configuration "
"will not be changed."
msgstr ""

#: pyanaconda/cmdline.py:56
msgid "Completed"
msgstr ""

#: pyanaconda/cmdline.py:64
msgid "In progress"
msgstr ""

#: pyanaconda/cmdline.py:77 pyanaconda/cmdline.py:110
msgid ""
"Command line mode requires all choices to be specified in a kickstart "
"configuration file."
msgstr ""

#: pyanaconda/cmdline.py:98 pyanaconda/gui.py:1203
#: pyanaconda/kickstart.py:1218 pyanaconda/kickstart.py:1298
#: pyanaconda/kickstart.py:1306 pyanaconda/kickstart.py:1335
#: pyanaconda/kickstart.py:1345 pyanaconda/text.py:421
#, python-format
msgid ""
"The following error was found while parsing the kickstart configuration "
"file:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/cmdline.py:129 pyanaconda/cmdline.py:136
#: pyanaconda/cmdline.py:143 pyanaconda/cmdline.py:150
msgid "Can't have a question in command line mode!"
msgstr ""

#: pyanaconda/constants.py:68
msgid ""
"An unhandled exception has occurred.  This is most likely a bug.  Please "
"save a copy of the detailed exception and file a bug report"
msgstr ""

#: pyanaconda/constants.py:74
msgid " with the provider of this software."
msgstr ""

#: pyanaconda/constants.py:78
#, python-format
msgid " against anaconda at %s"
msgstr ""

#: pyanaconda/gui.py:109
msgid "An error occurred saving screenshots to disk."
msgstr ""

#: pyanaconda/gui.py:126
msgid "Screenshots Copied"
msgstr ""

#: pyanaconda/gui.py:127
msgid ""
"The screenshots have been saved in the directory:\n"
"\n"
"\t/root/anaconda-screenshots/\n"
"\n"
"You can access these when you reboot and login as root."
msgstr ""

#: pyanaconda/gui.py:166
msgid "Saving Screenshot"
msgstr ""

#: pyanaconda/gui.py:167
#, python-format
msgid "A screenshot named '%s' has been saved."
msgstr ""

#: pyanaconda/gui.py:170
msgid "Error Saving Screenshot"
msgstr ""

#: pyanaconda/gui.py:171
msgid ""
"An error occurred while saving the screenshot.  If this occurred during "
"package installation, you may need to try several times for it to succeed."
msgstr ""

#: pyanaconda/gui.py:521 pyanaconda/text.py:150
msgid ""
"Choose a passphrase for the encrypted devices. You will be prompted for this "
"passphrase during system boot."
msgstr ""

#: pyanaconda/gui.py:540 pyanaconda/gui.py:548 pyanaconda/text.py:188
#: pyanaconda/text.py:198
msgid "Error with passphrase"
msgstr ""

#: pyanaconda/gui.py:541 pyanaconda/text.py:189
msgid "The passphrases you entered were different.  Please try again."
msgstr ""

#: pyanaconda/gui.py:549
msgid "The passphrase must be at least eight characters long."
msgstr ""

#: pyanaconda/gui.py:583 pyanaconda/text.py:226
#, python-format
msgid ""
"Device %s is encrypted. In order to access the device's contents during "
"installation you must enter the device's passphrase below."
msgstr ""

#: pyanaconda/gui.py:675 pyanaconda/gui.py:1410
#: pyanaconda/partIntfHelpers.py:159 pyanaconda/partIntfHelpers.py:344
#: pyanaconda/text.py:101 pyanaconda/text.py:102
#: pyanaconda/iw/account_gui.py:131 loader/dirbrowser.c:137
#: loader/driverdisk.c:393 loader/kickstart.c:373 loader/loader.c:458
#: loader/loader.c:570 pyanaconda/textw/constants_text.py:48
msgid "Cancel"
msgstr ""

#: pyanaconda/gui.py:687 pyanaconda/gui.py:688 pyanaconda/gui.py:784
#: pyanaconda/gui.py:856 pyanaconda/gui.py:857 data/ui/anaconda.glade.h:3
msgid "_Debug"
msgstr ""

#: pyanaconda/gui.py:983
msgid "Wireless setup"
msgstr ""

#: pyanaconda/gui.py:984
msgid "Scanning access points for wireless devices"
msgstr ""

#: pyanaconda/gui.py:1012 pyanaconda/textw/netconfig_text.py:399
msgid "Waiting for NetworkManager"
msgstr ""

#: pyanaconda/gui.py:1013
#, python-format
msgid "Waiting for NetworkManager to activate these devices: %s"
msgstr ""

#: pyanaconda/gui.py:1040
#, python-format
msgid "Failed to activate these network interfaces: %s"
msgstr ""

#: pyanaconda/gui.py:1043
msgid "Network Configuration"
msgstr ""

#: pyanaconda/gui.py:1052
#, python-format
msgid "An error occurred trying to bring up the %s network interface."
msgstr ""

#: pyanaconda/gui.py:1054
msgid "Error Enabling Network"
msgstr ""

#: pyanaconda/gui.py:1149 pyanaconda/text.py:441
msgid ""
"You do not have an active network connection.  This is required by some "
"exception saving methods.  Would you like to configure your network now?"
msgstr ""

#: pyanaconda/gui.py:1155 pyanaconda/rescue.py:255 pyanaconda/text.py:447
#: pyanaconda/yuminstall.py:819 pyanaconda/yuminstall.py:1241
#: pyanaconda/iw/task_gui.py:308
msgid "No Network Available"
msgstr ""

#: pyanaconda/gui.py:1156 pyanaconda/text.py:448
msgid "Remote exception saving methods will not work."
msgstr ""

#: pyanaconda/gui.py:1205 pyanaconda/text.py:423
msgid "Error Parsing Kickstart Config"
msgstr ""

#: pyanaconda/gui.py:1233
msgid "default:LTR"
msgstr ""

#: pyanaconda/gui.py:1314 pyanaconda/text.py:542
msgid "Error!"
msgstr ""

#: pyanaconda/gui.py:1315 pyanaconda/text.py:543
#, python-format
msgid ""
"An error occurred when attempting to load an installer interface component.\n"
"\n"
"className = %s"
msgstr ""

#: pyanaconda/gui.py:1320 pyanaconda/image.py:145 pyanaconda/image.py:173
#: pyanaconda/packages.py:337 pyanaconda/storage/__init__.py:1851
msgid "_Exit"
msgstr ""

#: pyanaconda/gui.py:1321 pyanaconda/image.py:145 pyanaconda/image.py:173
#: pyanaconda/livecd.py:167 pyanaconda/yuminstall.py:910
#: pyanaconda/yuminstall.py:1261 pyanaconda/yuminstall.py:1316
#: pyanaconda/yuminstall.py:1526
msgid "_Retry"
msgstr ""

#: pyanaconda/gui.py:1323 pyanaconda/storage/partitioning.py:283
msgid "The system will now reboot."
msgstr ""

#: pyanaconda/gui.py:1324 pyanaconda/packages.py:341
msgid "_Reboot"
msgstr ""

#: pyanaconda/gui.py:1326
msgid "Exiting"
msgstr ""

#: pyanaconda/gui.py:1407 pyanaconda/livecd.py:126 pyanaconda/text.py:292
#: pyanaconda/upgrade.py:242
msgid "Exit installer"
msgstr ""

#: pyanaconda/gui.py:1408
msgid "Are you sure you wish to exit the installer?"
msgstr ""

#: pyanaconda/gui.py:1417
#, python-format
msgid "%s Installer"
msgstr ""

#: pyanaconda/gui.py:1423
msgid "Unable to load title bar"
msgstr ""

#: pyanaconda/gui.py:1498
msgid "Install Window"
msgstr ""

#: pyanaconda/image.py:74
#, python-format
msgid ""
"The ISO image %s has a size which is not a multiple of 2048 bytes.  This may "
"mean it was corrupted on transfer to this computer.\n"
"\n"
"It is recommended that you exit and abort your installation, but you can "
"choose to continue if you think this is in error."
msgstr ""

#: pyanaconda/image.py:137
msgid "Couldn't Mount ISO Source"
msgstr ""

#: pyanaconda/image.py:138
#, python-format
msgid ""
"An error occurred mounting the source device %s.  This may happen if your "
"ISO images are located on an advanced storage device like LVM or RAID, or if "
"there was a problem mounting a partition.  Click exit to abort the "
"installation."
msgstr ""

#: pyanaconda/image.py:164
msgid "Missing ISO 9660 Image"
msgstr ""

#: pyanaconda/image.py:165
msgid ""
"The installer has tried to mount the installation image, but cannot find it "
"on the hard drive.\n"
"\n"
"Please copy this image to the drive and click Retry.  Click Exit to abort "
"the installation."
msgstr ""

#: pyanaconda/image.py:214 pyanaconda/livecd.py:160
#: pyanaconda/partIntfHelpers.py:239 pyanaconda/text.py:384
#: pyanaconda/text.py:388 pyanaconda/yuminstall.py:422
#: pyanaconda/yuminstall.py:913 pyanaconda/yuminstall.py:1000
#: pyanaconda/yuminstall.py:1005 pyanaconda/yuminstall.py:1266
#: pyanaconda/yuminstall.py:1326 pyanaconda/yuminstall.py:1520
#: pyanaconda/yuminstall.py:1542 pyanaconda/iw/advanced_storage.py:371
#: pyanaconda/iw/advanced_storage.py:381 pyanaconda/iw/advanced_storage.py:431
#: pyanaconda/iw/autopart_type.py:95 pyanaconda/iw/cleardisks_gui.py:44
#: pyanaconda/iw/cleardisks_gui.py:54 pyanaconda/iw/filter_gui.py:432
#: pyanaconda/iw/lvm_dialog_gui.py:792 pyanaconda/iw/osbootwidget.py:211
#: pyanaconda/iw/osbootwidget.py:217 pyanaconda/iw/osbootwidget.py:226
#: pyanaconda/iw/partition_dialog_gui.py:383
#: pyanaconda/iw/raid_dialog_gui.py:226 pyanaconda/iw/raid_dialog_gui.py:352
#: pyanaconda/iw/task_gui.py:52 pyanaconda/iw/task_gui.py:150
#: pyanaconda/iw/task_gui.py:302 pyanaconda/iw/task_gui.py:464
#: loader/windows.h:30 loader/cdinstall.c:186 loader/cdinstall.c:416
#: loader/driverdisk.c:208 loader/driverdisk.c:455 loader/driverdisk.c:490
#: loader/driverdisk.c:518 loader/driverdisk.c:526 loader/driverdisk.c:598
#: loader/hdinstall.c:282 loader/hdinstall.c:300 loader/hdinstall.c:309
#: loader/hdinstall.c:344 loader/hdinstall.c:383 loader/hdinstall.c:398
#: loader/kickstart.c:308 loader/lang.c:113 loader/loader.c:422
#: loader/loader.c:533 loader/loader.c:1216 loader/mediacheck.c:47
#: loader/mediacheck.c:86 loader/mediacheck.c:93 loader/mediacheck.c:97
#: loader/method.c:194 loader/method.c:268 loader/nfsinstall.c:255
#: loader/urlinstall.c:139 loader/urlinstall.c:161 loader/urlinstall.c:213
#: loader/urls.c:335 loader/urls.c:343 loader/urls.c:350
#: pyanaconda/storage/__init__.py:247 pyanaconda/storage/__init__.py:1852
#: pyanaconda/storage/__init__.py:1939 pyanaconda/storage/zfcp.py:375
#: pyanaconda/textw/upgrade_text.py:181 pyanaconda/textw/upgrade_text.py:188
msgid "Error"
msgstr ""

#: pyanaconda/image.py:215
#, python-format
msgid ""
"An error occurred unmounting the disc.  Please make sure you're not "
"accessing %s from the shell on tty2 and then click OK to retry."
msgstr ""

#: pyanaconda/installclass.py:68
msgid "Install on System"
msgstr ""

#: pyanaconda/installinterfacebase.py:52
#, python-format
msgid ""
"Disk %s contains BIOS RAID metadata, but is not part of any recognized BIOS "
"RAID sets. Ignoring disk %s."
msgid_plural ""
"Disks %s contain BIOS RAID metadata, but are not part of any recognized BIOS "
"RAID sets. Ignoring disks %s."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:82 data/ui/reinitialize-dialog.glade.h:3
msgid "Storage Device Warning"
msgstr ""

#: pyanaconda/installinterfacebase.py:131
#, python-format
msgid ""
"Error processing LVM.\n"
"There is inconsistent LVM data on %(msg)s.  You can reinitialize all related "
"PVs (%(pvs)s) which will erase the LVM metadata, or ignore which will "
"preserve the contents.  This action may also be applied to all other PVs "
"with inconsistent metadata."
msgstr ""

#: pyanaconda/installinterfacebase.py:138
#: pyanaconda/installinterfacebase.py:172
msgid "_Ignore"
msgstr ""

#: pyanaconda/installinterfacebase.py:139
msgid "Ignore _all"
msgstr ""

#: pyanaconda/installinterfacebase.py:140
msgid "_Re-initialize"
msgstr ""

#: pyanaconda/installinterfacebase.py:141
msgid "Re-ini_tialize all"
msgstr ""

#: pyanaconda/installinterfacebase.py:159
msgid "Unformatted DASD Device Found"
msgid_plural "Unformatted DASD Devices Found"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:161
#, python-format
msgid ""
"Format uninitialized DASD device?\n"
"\n"
"There is %d uninitialized DASD device on this system.  To continue "
"installation, the device must be formatted.  Formatting will remove any data "
"on this device."
msgid_plural ""
"Format uninitialized DASD devices?\n"
"\n"
"There are %d uninitialized DASD devices on this system.  To continue "
"installation, the devices must be formatted.  Formatting will remove any "
"data on these devices."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/installinterfacebase.py:172 pyanaconda/iw/partition_gui.py:901
msgid "_Format"
msgstr ""

#: pyanaconda/iutil.py:830
msgid "Error determining boot device's disk name"
msgstr ""

#: pyanaconda/iutil.py:852
msgid "the device containing /boot"
msgstr ""

#: pyanaconda/iutil.py:854
#, python-format
msgid ""
"After shutdown, please perform a manual IPL from %s to continue installation."
msgstr ""

#: pyanaconda/kickstart.py:116
#, python-format
msgid ""
"There was an error running the kickstart script at line %(lineno)s.  You may "
"examine the output in %(msgs)s.  This is a fatal error and installation will "
"be aborted.  Press the OK button to exit the installer."
msgstr ""

#: pyanaconda/kickstart.py:124 pyanaconda/kickstart.py:126
msgid "Scriptlet Failure"
msgstr ""

#: pyanaconda/kickstart.py:150
#, python-format
msgid ""
"Escrow certificate with url %s requires network to be enabled in loader or "
"configured in kickstart file."
msgstr ""

#: pyanaconda/kickstart.py:164
#, python-format
msgid ""
"The following error was encountered while downloading the escrow "
"certificate:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/kickstart.py:409
msgid "Could not log into any iSCSI nodes at the portal."
msgstr ""

#: pyanaconda/kickstart.py:1368 pyanaconda/livecd.py:190
msgid "Post-Installation"
msgstr ""

#: pyanaconda/kickstart.py:1369
msgid "Running post-installation scripts"
msgstr ""

#: pyanaconda/kickstart.py:1384
msgid "Running pre-installation scripts"
msgstr ""

#: pyanaconda/kickstart.py:1415
msgid "Missing Package"
msgstr ""

#: pyanaconda/kickstart.py:1416
#, python-format
msgid ""
"You have specified that the package '%s' should be installed.  This package "
"does not exist. Would you like to continue or abort this installation?"
msgstr ""

#: pyanaconda/kickstart.py:1422 pyanaconda/kickstart.py:1461
msgid "_Abort"
msgstr ""

#: pyanaconda/kickstart.py:1423 pyanaconda/kickstart.py:1462
msgid "_Ignore All"
msgstr ""

#: pyanaconda/kickstart.py:1453
msgid "Missing Group"
msgstr ""

#: pyanaconda/kickstart.py:1454
#, python-format
msgid ""
"You have specified that the group '%s' should be installed. This group does "
"not exist. Would you like to continue or abort this installation?"
msgstr ""

#: pyanaconda/kickstart.py:1551
#, python-format
msgid ""
"The kickstart configuration file is missing required information that "
"anaconda cannot prompt for.  Please add the following sections and try "
"again:\n"
"%s"
msgstr ""

#: pyanaconda/livecd.py:121
msgid "Unable to find image"
msgstr ""

#: pyanaconda/livecd.py:122
#, python-format
msgid ""
"The given location isn't a valid %s live CD to use as an installation source."
msgstr ""

#: pyanaconda/livecd.py:143
msgid "Copying live image to hard drive."
msgstr ""

#: pyanaconda/livecd.py:161
msgid ""
"There was an error installing the live image to your hard drive.  This could "
"be due to bad media.  Please verify your installation media.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"require reinstallation."
msgstr ""

#: pyanaconda/livecd.py:191
msgid ""
"Performing post-installation filesystem changes.  This may take several "
"minutes."
msgstr ""

#: pyanaconda/network.py:78
msgid "Hostname must be 255 or fewer characters in length."
msgstr ""

#: pyanaconda/network.py:84
msgid ""
"Hostname must start with a valid character in the ranges 'a-z', 'A-Z', or "
"'0-9'"
msgstr ""

#: pyanaconda/network.py:89
msgid ""
"Hostnames can only contain the characters 'a-z', 'A-Z', '0-9', '-', or '.'"
msgstr ""

#: pyanaconda/network.py:133
msgid "IP address is missing."
msgstr ""

#: pyanaconda/network.py:137
msgid ""
"IPv4 addresses must contain four numbers between 0 and 255, separated by "
"periods."
msgstr ""

#: pyanaconda/network.py:140
#, python-format
msgid "'%s' is not a valid IPv6 address."
msgstr ""

#: pyanaconda/network.py:142
#, python-format
msgid "'%s' is an invalid IP address."
msgstr ""

#: pyanaconda/packages.py:118
msgid "Resizing Failed"
msgstr ""

#: pyanaconda/packages.py:119
#, python-format
msgid "There was an error encountered while resizing the device %s."
msgstr ""

#: pyanaconda/packages.py:127
msgid "Migration Failed"
msgstr ""

#: pyanaconda/packages.py:128
#, python-format
msgid "An error was encountered while migrating filesystem on device %s."
msgstr ""

#: pyanaconda/packages.py:137
msgid "_File Bug"
msgstr ""

#: pyanaconda/packages.py:321
#, python-format
msgid ""
"Warning!  This is pre-release software!\n"
"\n"
"Thank you for downloading this pre-release of %(productName)s.\n"
"\n"
"This is not a final release and is not intended for use on production "
"systems.  The purpose of this release is to collect feedback from testers, "
"and it is not suitable for day to day usage.\n"
"\n"
"To report feedback, please visit:\n"
"\n"
"   %(bugzillaUrl)s\n"
"\n"
"and file a report against '%(fileagainst)s'.\n"
msgstr ""

#: pyanaconda/packages.py:337
msgid "_Install Anyway"
msgstr ""

#: pyanaconda/packages.py:340
msgid "Your system will now be rebooted..."
msgstr ""

#: pyanaconda/packages.py:341 pyanaconda/upgrade.py:250
#: pyanaconda/yuminstall.py:1007 pyanaconda/yuminstall.py:1092
#: pyanaconda/yuminstall.py:1507 pyanaconda/yuminstall.py:1549
#: data/ui/anaconda.glade.h:2
msgid "_Back"
msgstr ""

#: pyanaconda/packages.py:342
msgid "Warning! This is pre-release software!"
msgstr ""

#: pyanaconda/partIntfHelpers.py:47
msgid "Please enter a volume group name."
msgstr ""

#: pyanaconda/partIntfHelpers.py:51
msgid "Volume Group Names must be less than 128 characters"
msgstr ""

#: pyanaconda/partIntfHelpers.py:54
#, python-format
msgid "Error - the volume group name %s is not valid."
msgstr ""

#: pyanaconda/partIntfHelpers.py:59
msgid ""
"Error - the volume group name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: pyanaconda/partIntfHelpers.py:69
msgid "Please enter a logical volume name."
msgstr ""

#: pyanaconda/partIntfHelpers.py:73
msgid "Logical Volume Names must be less than 128 characters"
msgstr ""

#: pyanaconda/partIntfHelpers.py:77
#, python-format
msgid "Error - the logical volume name %s is not valid."
msgstr ""

#: pyanaconda/partIntfHelpers.py:83
msgid ""
"Error - the logical volume name contains illegal characters or spaces.  "
"Acceptable characters are letters, digits, '.' or '_'."
msgstr ""

#: pyanaconda/partIntfHelpers.py:111
#, python-format
msgid ""
"The mount point %s is invalid.  Mount points must start with '/' and cannot "
"end with '/', and must contain printable characters and no spaces."
msgstr ""

#: pyanaconda/partIntfHelpers.py:123 pyanaconda/partIntfHelpers.py:130
msgid "Unable To Delete"
msgstr ""

#: pyanaconda/partIntfHelpers.py:124
msgid "You must first select a partition to delete."
msgstr ""

#: pyanaconda/partIntfHelpers.py:155 pyanaconda/partIntfHelpers.py:343
#: pyanaconda/iw/lvm_dialog_gui.py:888
msgid "Confirm Delete"
msgstr ""

#: pyanaconda/partIntfHelpers.py:156
#, python-format
msgid "You are about to delete all partitions on the device '%s'."
msgstr ""

#: pyanaconda/partIntfHelpers.py:159 pyanaconda/partIntfHelpers.py:344
#: pyanaconda/iw/lvm_dialog_gui.py:891 pyanaconda/iw/lvm_dialog_gui.py:1491
#: pyanaconda/iw/osbootwidget.py:104 pyanaconda/iw/partition_gui.py:1815
#: pyanaconda/iw/partition_gui.py:1827
msgid "_Delete"
msgstr ""

#: pyanaconda/partIntfHelpers.py:206
msgid "Notice"
msgstr ""

#: pyanaconda/partIntfHelpers.py:207
#, python-format
msgid ""
"The following partitions were not deleted because they are in use:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/partIntfHelpers.py:222
msgid "Format as Swap?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:223
#, python-format
msgid ""
"%s has a partition type of 0x82 (Linux swap) but does not appear to be "
"formatted as a Linux swap partition.\n"
"\n"
"Would you like to format this partition as a swap partition?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:238
#, python-format
msgid "You need to select at least one hard drive to install %s."
msgstr ""

#: pyanaconda/partIntfHelpers.py:243
msgid ""
"You have chosen to use a pre-existing partition for this installation "
"without formatting it. We recommend that you format this partition to make "
"sure files from a previous operating system installation do not cause "
"problems with this installation of Linux. However, if this partition "
"contains files that you need to keep, such as home directories, then "
"continue without formatting this partition."
msgstr ""

#: pyanaconda/partIntfHelpers.py:251
msgid "Format?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:251 pyanaconda/iw/partition_gui.py:1600
msgid "_Modify Partition"
msgstr ""

#: pyanaconda/partIntfHelpers.py:251
msgid "Do _Not Format"
msgstr ""

#: pyanaconda/partIntfHelpers.py:259
msgid "Error with Partitioning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:260
#, python-format
msgid ""
"The following critical errors exist with your requested partitioning scheme. "
"These errors must be corrected prior to continuing with your install of "
"%(productName)s.\n"
"\n"
"%(errorstr)s"
msgstr ""

#: pyanaconda/partIntfHelpers.py:276
msgid "Partitioning Warning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:277
#, python-format
msgid ""
"The following warnings exist with your requested partition scheme.\n"
"\n"
"%s\n"
"\n"
"Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:291
msgid ""
"The following pre-existing partitions have been selected to be formatted, "
"destroying all data."
msgstr ""

#: pyanaconda/partIntfHelpers.py:294
msgid ""
"Select 'Yes' to continue and format these partitions, or 'No' to go back and "
"change these settings."
msgstr ""

#: pyanaconda/partIntfHelpers.py:300
msgid "Format Warning"
msgstr ""

#: pyanaconda/partIntfHelpers.py:327
#, python-format
msgid ""
"You are about to delete the volume group \"%s\".\n"
"\n"
"ALL logical volumes in this volume group will be lost!"
msgstr ""

#: pyanaconda/partIntfHelpers.py:331
#, python-format
msgid "You are about to delete the logical volume \"%s\"."
msgstr ""

#: pyanaconda/partIntfHelpers.py:334
msgid "You are about to delete a RAID device."
msgstr ""

#: pyanaconda/partIntfHelpers.py:336
#, python-format
msgid "You are about to delete the %s partition."
msgstr ""

#: pyanaconda/partIntfHelpers.py:340
#, python-format
msgid "You are about to delete the %(type)s %(name)s"
msgstr ""

#: pyanaconda/partIntfHelpers.py:351
msgid "Confirm Reset"
msgstr ""

#: pyanaconda/partIntfHelpers.py:352
msgid ""
"Are you sure you want to reset the partition table to its original state?"
msgstr ""

#: pyanaconda/partIntfHelpers.py:361
msgid "No credentials (discovery authentication disabled)"
msgstr ""

#: pyanaconda/partIntfHelpers.py:362
msgid "CHAP pair"
msgstr ""

#: pyanaconda/partIntfHelpers.py:363
msgid "CHAP pair and a reverse pair"
msgstr ""

#: pyanaconda/partIntfHelpers.py:364
msgid "Use the credentials from the discovery step"
msgstr ""

#: pyanaconda/partIntfHelpers.py:472
msgid "iSCSI Nodes"
msgstr ""

#: pyanaconda/partIntfHelpers.py:473
msgid "No new iSCSI nodes discovered"
msgstr ""

#: pyanaconda/partIntfHelpers.py:514
msgid "iSCSI Error"
msgstr ""

#: pyanaconda/partIntfHelpers.py:519
msgid "Discovery"
msgstr ""

#: pyanaconda/partIntfHelpers.py:519
msgid "Login"
msgstr ""

#: pyanaconda/partIntfHelpers.py:520
#, python-format
msgid "iSCSI %s Error"
msgstr ""

#: pyanaconda/platform.py:99 pyanaconda/platform.py:351
#: pyanaconda/platform.py:419 pyanaconda/platform.py:541
#: data/ui/create-storage.glade.h:15
msgid "RAID Device"
msgstr ""

#: pyanaconda/platform.py:100 pyanaconda/platform.py:103
#: pyanaconda/platform.py:352 pyanaconda/platform.py:420
#: pyanaconda/platform.py:542 pyanaconda/platform.py:545
msgid "Master Boot Record (MBR)"
msgstr ""

#: pyanaconda/platform.py:102 pyanaconda/platform.py:544
msgid "First sector of boot partition"
msgstr ""

#: pyanaconda/platform.py:114
msgid "You have not created a bootable partition."
msgstr ""

#: pyanaconda/platform.py:119
msgid "Bootable partitions cannot be on a RAID device."
msgstr ""

#: pyanaconda/platform.py:121
msgid "Bootable partitions can only be on RAID1 devices."
msgstr ""

#: pyanaconda/platform.py:125
msgid "Bootable RAID1 set members must be partitions."
msgstr ""

#: pyanaconda/platform.py:130
msgid "Bootable partitions cannot be on a logical volume."
msgstr ""

#: pyanaconda/platform.py:137
#, python-format
msgid "Bootable partitions cannot be on an %s filesystem."
msgstr ""

#: pyanaconda/platform.py:141 pyanaconda/platform.py:146
msgid "Bootable partitions cannot be on an encrypted block device"
msgstr ""

#: pyanaconda/platform.py:230
msgid "EFI System Partition"
msgstr ""

#: pyanaconda/platform.py:242
msgid "You have not created a /boot/efi partition."
msgstr ""

#: pyanaconda/platform.py:248
msgid "/boot/efi is not EFI."
msgstr ""

#: pyanaconda/platform.py:264
#, python-format
msgid "%s must have a %s disk label."
msgstr ""

#: pyanaconda/platform.py:296
#, python-format
msgid "%s must have a bsd disk label."
msgstr ""

#: pyanaconda/platform.py:308
#, python-format
msgid "The disk %s requires at least 1MB of free space at the beginning."
msgstr ""

#: pyanaconda/platform.py:354
msgid "PPC PReP Boot"
msgstr ""

#: pyanaconda/platform.py:370
msgid "The boot partition must be within the first 4MB of the disk."
msgstr ""

#: pyanaconda/platform.py:422 pyanaconda/platform.py:425
msgid "Apple Bootstrap"
msgstr ""

#: pyanaconda/platform.py:439
#, python-format
msgid "%s must have a mac disk label."
msgstr ""

#: pyanaconda/rescue.py:211 pyanaconda/rescue.py:367 pyanaconda/rescue.py:467
#, python-format
msgid "Run %s to unmount the system when you are finished."
msgstr ""

#: pyanaconda/rescue.py:214
msgid "When finished please exit from the shell and your system will reboot."
msgstr ""

#: pyanaconda/rescue.py:228
msgid "Unable to find /bin/sh to execute!  Not starting shell"
msgstr ""

#: pyanaconda/rescue.py:247
msgid "Setup Networking"
msgstr ""

#: pyanaconda/rescue.py:248
msgid "Do you want to start the network interfaces on this system?"
msgstr ""

#: pyanaconda/rescue.py:249 loader/driverdisk.c:638 loader/driverdisk.c:648
#: loader/hdinstall.c:168 pyanaconda/textw/constants_text.py:56
msgid "Yes"
msgstr ""

#: pyanaconda/rescue.py:249 pyanaconda/rescue.py:251 loader/driverdisk.c:638
#: loader/driverdisk.c:648 pyanaconda/textw/constants_text.py:60
msgid "No"
msgstr ""

#: pyanaconda/rescue.py:256
msgid ""
"Unable to activate a networking device.  Networking will not be available in "
"rescue mode."
msgstr ""

#: pyanaconda/rescue.py:292 pyanaconda/rescue.py:374 pyanaconda/rescue.py:384
#: pyanaconda/rescue.py:470 pyanaconda/rescue.py:503
msgid "Rescue"
msgstr ""

#: pyanaconda/rescue.py:293
#, python-format
msgid ""
"The rescue environment will now attempt to find your Linux installation and "
"mount it under the directory %s.  You can then make any changes required to "
"your system.  If you want to proceed with this step choose 'Continue'.  You "
"can also choose to mount your file systems read-only instead of read-write "
"by choosing 'Read-Only'.  If you need to activate SAN devices choose "
"'Advanced'.\n"
"\n"
"If for some reason this process fails you can choose 'Skip' and this step "
"will be skipped and you will go directly to a command shell.\n"
"\n"
msgstr ""

#: pyanaconda/rescue.py:304 pyanaconda/iw/partition_gui.py:788
#: loader/driverdisk.c:599 pyanaconda/storage/__init__.py:168
#: pyanaconda/storage/devicetree.py:95
msgid "Continue"
msgstr ""

#: pyanaconda/rescue.py:304 pyanaconda/rescue.py:313
msgid "Read-Only"
msgstr ""

#: pyanaconda/rescue.py:304 pyanaconda/rescue.py:306 loader/cdinstall.c:210
#: loader/cdinstall.c:213 loader/method.c:241
#: pyanaconda/storage/__init__.py:1851 pyanaconda/textw/upgrade_text.py:139
msgid "Skip"
msgstr ""

#: pyanaconda/rescue.py:304 pyanaconda/rescue.py:309
msgid "Advanced"
msgstr ""

#: pyanaconda/rescue.py:343
msgid "System to Rescue"
msgstr ""

#: pyanaconda/rescue.py:344
msgid "Which device holds the root partition of your installation?"
msgstr ""

#: pyanaconda/rescue.py:346 pyanaconda/rescue.py:350 pyanaconda/text.py:547
#: pyanaconda/text.py:549
msgid "Exit"
msgstr ""

#: pyanaconda/rescue.py:364 pyanaconda/rescue.py:464
msgid "The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:375
#, python-format
msgid ""
"Your system had dirty file systems which you chose not to mount.  Press "
"return to get a shell from which you can fsck and mount your partitions. %s"
msgstr ""

#: pyanaconda/rescue.py:385
#, python-format
msgid ""
"Your system has been mounted under %(rootPath)s.\n"
"\n"
"Press <return> to get a shell. If you would like to make your system the "
"root environment, run the command:\n"
"\n"
"\tchroot %(rootPath)s\n"
"\n"
"%(msg)s"
msgstr ""

#: pyanaconda/rescue.py:471
#, python-format
msgid ""
"An error occurred trying to mount some or all of your system. Some of it may "
"be mounted under %s.\n"
"\n"
"Press <return> to get a shell. %s"
msgstr ""

#: pyanaconda/rescue.py:481
msgid "You don't have any Linux partitions.  Rebooting.\n"
msgstr ""

#: pyanaconda/rescue.py:485
msgid " The system will reboot automatically when you exit from the shell."
msgstr ""

#: pyanaconda/rescue.py:489
msgid "Rescue Mode"
msgstr ""

#: pyanaconda/rescue.py:490
#, python-format
msgid "You don't have any Linux partitions. Press return to get a shell.%s"
msgstr ""

#: pyanaconda/rescue.py:502
#, python-format
msgid "Your system is mounted under the %s directory."
msgstr ""

#: pyanaconda/text.py:155
msgid "Passphrase for encrypted device"
msgstr ""

#: pyanaconda/text.py:168
msgid "Also add this passphrase to all existing encrypted devices"
msgstr ""

#: pyanaconda/text.py:199
#, python-format
msgid "The passphrase must be at least %d character long."
msgid_plural "The passphrase must be at least %d characters long."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/text.py:233 data/ui/lukspassphrase.glade.h:6
msgid "Passphrase"
msgstr ""

#: pyanaconda/text.py:241 data/ui/lukspassphrase.glade.h:7
msgid "This is a global passphrase"
msgstr ""

#: pyanaconda/text.py:274
msgid "Yes, discard any data"
msgstr ""

#: pyanaconda/text.py:275
msgid "No, keep any data"
msgstr ""

#: pyanaconda/text.py:385 pyanaconda/text.py:389
msgid "Repository editing is not available in text mode."
msgstr ""

#: pyanaconda/text.py:466
#, python-format
msgid "Welcome to %(productName)s for %(productArch)s"
msgstr ""

#: pyanaconda/text.py:468
#, python-format
msgid "Welcome to %s"
msgstr ""

#: pyanaconda/text.py:470
msgid ""
"  <Tab>/<Alt-Tab> between elements   |  <Space> selects   |  <F12> next "
"screen"
msgstr ""

#: pyanaconda/text.py:547 loader/net.c:109 loader/net.c:512 loader/net.c:570
#: loader/net.c:715 loader/net.c:723 loader/net.c:1151 loader/net.c:1159
#: pyanaconda/textw/constants_text.py:68
msgid "Retry"
msgstr ""

#: pyanaconda/text.py:561
msgid "Cancelled"
msgstr ""

#: pyanaconda/text.py:562
msgid "I can't go to the previous step from here. You will have to try again."
msgstr ""

#: pyanaconda/upgrade.py:51
msgid "Proceed with upgrade?"
msgstr ""

#: pyanaconda/upgrade.py:52
msgid ""
"The file systems of the Linux installation you have chosen to upgrade have "
"already been mounted. You cannot go back past this point. \n"
"\n"
msgstr ""

#: pyanaconda/upgrade.py:56
msgid "Would you like to continue with the upgrade?"
msgstr ""

#: pyanaconda/upgrade.py:91
msgid "Cannot Upgrade"
msgstr ""

#: pyanaconda/upgrade.py:92
#, python-format
msgid ""
"Your current installation cannot be upgraded. This is likely due to it being "
"too old. Only the previous two release may be upgraded. To upgrade older "
"releases you must first upgrade through all intermediate releases.\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/upgrade.py:229
msgid "Mount failed"
msgstr ""

#: pyanaconda/upgrade.py:230
#, python-format
msgid ""
"The following error occurred when mounting the file systems listed in /etc/"
"fstab.  Please fix this problem and try to upgrade again.\n"
"%s"
msgstr ""

#: pyanaconda/upgrade.py:238 pyanaconda/upgrade.py:245
msgid "Upgrade root not found"
msgstr ""

#: pyanaconda/upgrade.py:239
msgid "The root for the previously installed system was not found."
msgstr ""

#: pyanaconda/upgrade.py:246
msgid ""
"The root for the previously installed system was not found.  You can exit "
"installer or backtrack to choose installation instead of upgrade."
msgstr ""

#: pyanaconda/upgrade.py:269
msgid ""
"The following files are absolute symbolic links, which we do not support "
"during an upgrade. Please change them to relative symbolic links and restart "
"the upgrade.\n"
"\n"
msgstr ""

#: pyanaconda/upgrade.py:275
msgid "Absolute Symlinks"
msgstr ""

#: pyanaconda/upgrade.py:286
msgid ""
"The following are directories which should instead be symbolic links, which "
"will cause problems with the upgrade.  Please return them to their original "
"state as symbolic links and restart the upgrade.\n"
"\n"
msgstr ""

#: pyanaconda/upgrade.py:292
msgid "Invalid Directories"
msgstr ""

#: pyanaconda/vnc.py:136
#, python-format
msgid "%(productName)s %(productVersion)s installation on host %(name)s"
msgstr ""

#: pyanaconda/vnc.py:142
#, python-format
msgid "%(productName)s %(productVersion)s installation"
msgstr ""

#: pyanaconda/vnc.py:161
#, python-format
msgid "Attempting to connect to vnc client on host %s..."
msgstr ""

#: pyanaconda/vnc.py:175
msgid "Connected!"
msgstr ""

#: pyanaconda/vnc.py:178
msgid "Will try to connect again in 15 seconds..."
msgstr ""

#: pyanaconda/vnc.py:184
#, python-format
msgid "Giving up attempting to connect after %d try!\n"
msgid_plural "Giving up attempting to connect after %d tries!\n"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/vnc.py:195
#, python-format
msgid "Please manually connect your vnc client to %s to begin the install."
msgstr ""

#: pyanaconda/vnc.py:197
msgid "Please manually connect your vnc client to begin the install."
msgstr ""

#: pyanaconda/vnc.py:200
msgid "Starting VNC..."
msgstr ""

#: pyanaconda/vnc.py:237
msgid "The VNC server is now running."
msgstr ""

#: pyanaconda/vnc.py:241
msgid ""
"\n"
"\n"
"You chose to connect to a listening vncviewer. \n"
"This does not require a password to be set.  If you \n"
"set a password, it will be used in case the connection \n"
"to the vncviewer is unsuccessful\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:246
msgid ""
"\n"
"\n"
"WARNING!!! VNC server running with NO PASSWORD!\n"
"You can use the vncpassword=<password> boot option\n"
"if you would like to secure the server.\n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:250
msgid ""
"\n"
"\n"
"You chose to execute vnc with a password. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:252
msgid ""
"\n"
"\n"
"Unknown Error.  Aborting. \n"
"\n"
msgstr ""

#: pyanaconda/vnc.py:274 pyanaconda/vnc.py:367
msgid "VNC Configuration"
msgstr ""

#: pyanaconda/vnc.py:277 pyanaconda/vnc.py:371
msgid "No password"
msgstr ""

#: pyanaconda/vnc.py:279 pyanaconda/vnc.py:374
msgid ""
"A password will prevent unauthorized listeners connecting and monitoring "
"your installation progress.  Please enter a password to be used for the "
"installation"
msgstr ""

#: pyanaconda/vnc.py:287 pyanaconda/vnc.py:382
#: pyanaconda/textw/userauth_text.py:47
msgid "Password:"
msgstr ""

#: pyanaconda/vnc.py:288 pyanaconda/vnc.py:383
#: pyanaconda/textw/userauth_text.py:49
msgid "Password (confirm):"
msgstr ""

#: pyanaconda/vnc.py:306 pyanaconda/vnc.py:405
#: pyanaconda/textw/userauth_text.py:70
msgid "Password Mismatch"
msgstr ""

#: pyanaconda/vnc.py:307 pyanaconda/vnc.py:406
#: pyanaconda/textw/userauth_text.py:71
msgid "The passwords you entered were different. Please try again."
msgstr ""

#: pyanaconda/vnc.py:312 pyanaconda/vnc.py:411
#: pyanaconda/textw/userauth_text.py:66
msgid "Password Length"
msgstr ""

#: pyanaconda/vnc.py:313 pyanaconda/vnc.py:412
msgid "The password must be at least six characters long."
msgstr ""

#: pyanaconda/vnc.py:335
msgid "Unable to Start X"
msgstr ""

#: pyanaconda/vnc.py:337
msgid ""
"X was unable to start on your machine.  Would you like to start VNC to "
"connect to this computer from another computer and perform a graphical "
"install or continue with a text mode install?"
msgstr ""

#: pyanaconda/vnc.py:356
msgid "Start VNC"
msgstr ""

#: pyanaconda/vnc.py:357 pyanaconda/vnc.py:359
msgid "Use text mode"
msgstr ""

#: pyanaconda/yuminstall.py:85
#, python-format
msgid "%s MB"
msgstr ""

#: pyanaconda/yuminstall.py:88
#, python-format
msgid "%s KB"
msgstr ""

#: pyanaconda/yuminstall.py:90
#, python-format
msgid "%s Byte"
msgid_plural "%s Bytes"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/yuminstall.py:135
msgid "Preparing to install"
msgstr ""

#: pyanaconda/yuminstall.py:136
msgid "Preparing transaction from installation source"
msgstr ""

#: pyanaconda/yuminstall.py:166
#, python-format
msgid "<b>Upgrading %(pkgStr)s</b> (%(size)s)\n"
msgstr ""

#: pyanaconda/yuminstall.py:169
#, python-format
msgid "<b>Installing %(pkgStr)s</b> (%(size)s)\n"
msgstr ""

#: pyanaconda/yuminstall.py:219
#, python-format
msgid "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgid_plural "Packages completed: %(donepkgs)d of %(numpkgs)d"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/yuminstall.py:233
#, python-format
msgid "<b>Cleaning up %s</b>"
msgstr ""

#: pyanaconda/yuminstall.py:253
msgid "Error Installing Package"
msgstr ""

#: pyanaconda/yuminstall.py:254
#, python-format
msgid ""
"A fatal error occurred when installing the %s package.  This could indicate "
"errors when reading the installation media.  Installation cannot continue."
msgstr ""

#: pyanaconda/yuminstall.py:359 pyanaconda/iw/task_gui.py:320
msgid "Error Setting Up Repository"
msgstr ""

#: pyanaconda/yuminstall.py:360
#, python-format
msgid ""
"The following error occurred while setting up the installation repository:\n"
"\n"
"%(e)s\n"
"\n"
"Please provide the correct information for installing %(productName)s."
msgstr ""

#: pyanaconda/yuminstall.py:405
msgid "Change Disc"
msgstr ""

#: pyanaconda/yuminstall.py:406
#, python-format
msgid "Please insert the %(productName)s disc to continue."
msgstr ""

#: pyanaconda/yuminstall.py:415
msgid "Wrong Disc"
msgstr ""

#: pyanaconda/yuminstall.py:416
#, python-format
msgid "That's not the correct %s disc."
msgstr ""

#: pyanaconda/yuminstall.py:423
msgid "Unable to access the disc."
msgstr ""

#: pyanaconda/yuminstall.py:562
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr ""

#: pyanaconda/yuminstall.py:820 pyanaconda/yuminstall.py:1242
#: pyanaconda/iw/task_gui.py:309
msgid ""
"Some of your software repositories require networking, but there was an "
"error enabling the network on your system."
msgstr ""

#: pyanaconda/yuminstall.py:908 pyanaconda/yuminstall.py:910
msgid "Re_boot"
msgstr ""

#: pyanaconda/yuminstall.py:908
msgid "_Eject"
msgstr ""

#: pyanaconda/yuminstall.py:914
#, python-format
msgid ""
"The file %s cannot be opened.  This is due to a missing file, a corrupt "
"package or corrupt media.  Please verify your installation source.\n"
"\n"
"If you exit, your system will be left in an inconsistent state that will "
"likely require reinstallation.\n"
"\n"
msgstr ""

#: pyanaconda/yuminstall.py:961
msgid "Retrying"
msgstr ""

#: pyanaconda/yuminstall.py:961
msgid "Retrying download."
msgstr ""

#: pyanaconda/yuminstall.py:996
#, python-format
msgid ""
"There was an error running your transaction for the following reason: %s\n"
msgstr ""

#: pyanaconda/yuminstall.py:1038 pyanaconda/yuminstall.py:1039
msgid "file conflicts"
msgstr ""

#: pyanaconda/yuminstall.py:1040
msgid "older package(s)"
msgstr ""

#: pyanaconda/yuminstall.py:1041
msgid "insufficient disk space"
msgstr ""

#: pyanaconda/yuminstall.py:1042
msgid "insufficient disk inodes"
msgstr ""

#: pyanaconda/yuminstall.py:1043
msgid "package conflicts"
msgstr ""

#: pyanaconda/yuminstall.py:1044
msgid "package already installed"
msgstr ""

#: pyanaconda/yuminstall.py:1045
msgid "required package"
msgstr ""

#: pyanaconda/yuminstall.py:1046
msgid "package for incorrect arch"
msgstr ""

#: pyanaconda/yuminstall.py:1047
msgid "package for incorrect os"
msgstr ""

#: pyanaconda/yuminstall.py:1061
msgid "You need more space on the following file systems:\n"
msgstr ""

#: pyanaconda/yuminstall.py:1074
#, python-format
msgid ""
"There were file conflicts when checking the packages to be installed:\n"
"%s\n"
msgstr ""

#: pyanaconda/yuminstall.py:1077
#, python-format
msgid ""
"There was an error running your transaction for the following reason(s): "
"%s.\n"
msgstr ""

#: pyanaconda/yuminstall.py:1084 pyanaconda/yuminstall.py:1089
msgid "Error Running Transaction"
msgstr ""

#: pyanaconda/yuminstall.py:1267
msgid ""
"Unable to read group information from repositories.  This is a problem with "
"the generation of your install tree."
msgstr ""

#: pyanaconda/yuminstall.py:1316 pyanaconda/textw/constants_text.py:64
msgid "Edit"
msgstr ""

#: pyanaconda/yuminstall.py:1327 pyanaconda/yuminstall.py:1521
#, python-format
msgid ""
"Unable to read package metadata. This may be due to a missing repodata "
"directory.  Please ensure that your install tree has been correctly "
"generated.\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/yuminstall.py:1499
msgid ""
"Some of the packages you have selected for install are missing "
"dependencies.  You can exit the installation, go back and change your "
"package selections, or continue installing these packages without their "
"dependencies.  If you continue, these packages may not work correctly due to "
"missing components."
msgstr ""

#: pyanaconda/yuminstall.py:1543
#, python-format
msgid ""
"Your selected packages require %d MB of free space for installation, but you "
"do not have enough available.  You can change your selections or exit the "
"installer."
msgstr ""

#: pyanaconda/yuminstall.py:1624
#, python-format
msgid ""
"You appear to be upgrading from a system which is too old to upgrade to this "
"version of %s.  Are you sure you wish to continue the upgrade process?"
msgstr ""

#: pyanaconda/yuminstall.py:1659
#, python-format
msgid ""
"The arch of the release of %(productName)s you are upgrading to appears to "
"be %(myarch)s which does not match your previously installed arch of "
"%(arch)s.  This is likely to not succeed.  Are you sure you wish to continue "
"the upgrade process?"
msgstr ""

#: pyanaconda/yuminstall.py:1706
msgid "Post Upgrade"
msgstr ""

#: pyanaconda/yuminstall.py:1707
msgid "Performing post-upgrade configuration"
msgstr ""

#: pyanaconda/yuminstall.py:1709
msgid "Post Installation"
msgstr ""

#: pyanaconda/yuminstall.py:1710
msgid "Performing post-installation configuration"
msgstr ""

#: pyanaconda/yuminstall.py:1945
msgid "Installation Starting"
msgstr ""

#: pyanaconda/yuminstall.py:1946
msgid "Starting installation process"
msgstr ""

#: pyanaconda/yuminstall.py:1984
msgid "Dependency Check"
msgstr ""

#: pyanaconda/yuminstall.py:1985
msgid "Checking dependencies in packages selected for installation"
msgstr ""

#: pyanaconda/yuminstall.py:2041 pyanaconda/yuminstall.py:2057
msgid "Retrieving installation information."
msgstr ""

#: pyanaconda/yuminstall.py:2043 pyanaconda/yuminstall.py:2059
#, python-format
msgid "Retrieving installation information for %s."
msgstr ""

#: pyanaconda/yuminstall.py:2044 pyanaconda/yuminstall.py:2060
msgid "Installation Progress"
msgstr ""

#: pyanaconda/installclasses/fedora.py:38
msgid "_Fedora"
msgstr ""

#: pyanaconda/installclasses/fedora.py:39
#, python-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage. You can optionally select a different set of "
"software now."
msgstr ""

#: pyanaconda/installclasses/fedora.py:48
msgid "Graphical Desktop"
msgstr ""

#: pyanaconda/installclasses/fedora.py:53
msgid "Software Development"
msgstr ""

#: pyanaconda/installclasses/fedora.py:59
msgid "Web Server"
msgstr ""

#: pyanaconda/installclasses/fedora.py:63 pyanaconda/installclasses/rhel.py:44
msgid "Minimal"
msgstr ""

#: pyanaconda/installclasses/rhel.py:33
msgid "Red Hat Enterprise Linux"
msgstr ""

#: pyanaconda/installclasses/rhel.py:34
#, python-format
msgid ""
"The default installation of %s is a minimum install. You can optionally "
"select a different set of software now."
msgstr ""

#: pyanaconda/iw/GroupSelector.py:147
#, python-format
msgid "Packages in %s"
msgstr ""

#: pyanaconda/iw/GroupSelector.py:433
#, python-format
msgid "Optional packages selected: %(inst)d of %(cnt)d"
msgstr ""

#: pyanaconda/iw/GroupSelector.py:435
#, python-format
msgid "<i>%s</i>"
msgstr ""

#: pyanaconda/iw/GroupSelector.py:495
msgid "Uncategorized"
msgstr ""

#: pyanaconda/iw/account_gui.py:52
msgid "Root _Password:"
msgstr ""

#: pyanaconda/iw/account_gui.py:54
msgid "_Confirm:"
msgstr ""

#: pyanaconda/iw/account_gui.py:92
msgid "Caps Lock is on."
msgstr ""

#: pyanaconda/iw/account_gui.py:102 pyanaconda/iw/account_gui.py:110
#: pyanaconda/iw/account_gui.py:117 pyanaconda/iw/account_gui.py:138
#: pyanaconda/textw/userauth_text.py:74
msgid "Error with Password"
msgstr ""

#: pyanaconda/iw/account_gui.py:103
msgid ""
"You must enter your root password and confirm it by typing it a second time "
"to continue."
msgstr ""

#: pyanaconda/iw/account_gui.py:111
msgid "The passwords you entered were different.  Please try again."
msgstr ""

#: pyanaconda/iw/account_gui.py:118
msgid "The root password must be at least six characters long."
msgstr ""

#: pyanaconda/iw/account_gui.py:127 pyanaconda/textw/userauth_text.py:83
msgid "Weak Password"
msgstr ""

#: pyanaconda/iw/account_gui.py:128
#, python-format
msgid "You have provided a weak password: %s"
msgstr ""

#: pyanaconda/iw/account_gui.py:131
msgid "_Use Anyway"
msgstr ""

#: pyanaconda/iw/account_gui.py:139 pyanaconda/textw/userauth_text.py:75
msgid ""
"Requested password contains non-ASCII characters, which are not allowed."
msgstr ""

#: pyanaconda/iw/advanced_storage.py:249
msgid "Node Name"
msgstr ""

#: pyanaconda/iw/advanced_storage.py:372
msgid "You must select a NIC to use."
msgstr ""

#: pyanaconda/iw/autopart_type.py:96
msgid ""
"No partitions are available to resize.  Only physical partitions with "
"specific filesystems can be resized."
msgstr ""

#: pyanaconda/iw/autopart_type.py:121
msgid "Resize FileSystem Error"
msgstr ""

#: pyanaconda/iw/autopart_type.py:122
#, python-format
msgid "%(device)s: %(msg)s"
msgstr ""

#: pyanaconda/iw/autopart_type.py:131
msgid "Resize Device Error"
msgstr ""

#: pyanaconda/iw/autopart_type.py:132
#, python-format
msgid "%(name)s: %(msg)s"
msgstr ""

#: pyanaconda/iw/autopart_type.py:252
msgid "Use _All Space"
msgstr ""

#: pyanaconda/iw/autopart_type.py:254
msgid ""
"Removes all partitions on the selected device(s).  This includes partitions "
"created by other operating systems.\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: pyanaconda/iw/autopart_type.py:260
msgid "Replace Existing _Linux System(s)"
msgstr ""

#: pyanaconda/iw/autopart_type.py:262
msgid ""
"Removes all Linux partitions on the selected device(s). This does not remove "
"other partitions you may have on your storage device(s) (such as VFAT or "
"FAT32).\n"
"\n"
"<b>Tip:</b> This option will remove data from the selected device(s).  Make "
"sure you have backups."
msgstr ""

#: pyanaconda/iw/autopart_type.py:270
msgid "_Shrink Current System"
msgstr ""

#: pyanaconda/iw/autopart_type.py:272
msgid ""
"Shrinks existing partitions to create free space for the default layout."
msgstr ""

#: pyanaconda/iw/autopart_type.py:274
msgid "Use _Free Space"
msgstr ""

#: pyanaconda/iw/autopart_type.py:276
msgid ""
"Retains your current data and partitions and uses only the unpartitioned "
"space on the selected device(s), assuming you have enough free space "
"available."
msgstr ""

#: pyanaconda/iw/autopart_type.py:280
msgid "Create _Custom Layout"
msgstr ""

#: pyanaconda/iw/autopart_type.py:282
msgid ""
"Manually create your own custom layout on the selected device(s) using our "
"partitioning tool."
msgstr ""

#: pyanaconda/iw/blpasswidget.py:44
msgid "_Use a boot loader password"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:45
msgid ""
"A boot loader password prevents users from changing kernel options, "
"increasing security."
msgstr ""

#: pyanaconda/iw/blpasswidget.py:76
msgid "Change _password"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:99
msgid "Enter Boot Loader Password"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:105
msgid ""
"Enter a boot loader password and then confirm it.  (Note that your BIOS "
"keymap may be different than the actual keymap you are used to.)"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:112
msgid "_Password:"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:118
msgid "Con_firm:"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:140
msgid "Passwords don't match"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:141
msgid "Passwords do not match"
msgstr ""

#: pyanaconda/iw/blpasswidget.py:150
msgid ""
"Your boot loader password is shorter than six characters.  We recommend a "
"longer boot loader password.\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: pyanaconda/iw/bootloader_main_gui.py:36
msgid "Boot Loader Configuration"
msgstr ""

#: pyanaconda/iw/bootloader_main_gui.py:181
#: pyanaconda/iw/bootloader_main_gui.py:186
#: pyanaconda/iw/bootloader_main_gui.py:224
#, python-format
msgid "_Install boot loader on /dev/%s."
msgstr ""

#: pyanaconda/iw/bootloader_main_gui.py:234
msgid "_Change device"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:34
msgid "Clear Disks Selector"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:45 pyanaconda/iw/filter_gui.py:433
msgid "You must select at least one drive to be used for installation."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:55
msgid "You must select one drive to boot from."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:127 pyanaconda/iw/cleardisks_gui.py:147
#: pyanaconda/iw/filter_gui.py:490 pyanaconda/iw/filter_gui.py:501
#: pyanaconda/iw/filter_gui.py:531
msgid "Model"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:128 pyanaconda/iw/cleardisks_gui.py:148
#: pyanaconda/iw/filter_gui.py:491 pyanaconda/iw/filter_gui.py:502
#: pyanaconda/iw/filter_gui.py:510 pyanaconda/iw/filter_gui.py:521
#: pyanaconda/iw/filter_gui.py:532
msgid "Capacity"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:129 pyanaconda/iw/filter_gui.py:492
#: pyanaconda/iw/filter_gui.py:511 pyanaconda/iw/filter_gui.py:522
#: pyanaconda/iw/filter_gui.py:533
msgid "Vendor"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:130 pyanaconda/iw/cleardisks_gui.py:149
#: pyanaconda/iw/filter_gui.py:509 pyanaconda/iw/filter_gui.py:520
#: pyanaconda/iw/filter_gui.py:536
msgid "Identifier"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:131 pyanaconda/iw/filter_gui.py:493
#: pyanaconda/iw/filter_gui.py:512 pyanaconda/iw/filter_gui.py:523
#: pyanaconda/iw/filter_gui.py:534
msgid "Interconnect"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:145
msgid ""
"Boot\n"
"Loader"
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:196
msgid ""
"<b>Tip:</b> All Linux filesystems on the install target devices will be "
"reformatted and wiped of any data.  Make sure you have backups."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:198
msgid ""
"<b>Tip:</b> The install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: pyanaconda/iw/cleardisks_gui.py:200
msgid ""
"<b>Tip:</b> Your filesystems on the install target devices will not be "
"reformatted unless you choose to do so during customization."
msgstr ""

#: pyanaconda/iw/congrats_gui.py:34
msgid "Congratulations"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:75 pyanaconda/textw/complete_text.py:39
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:78 pyanaconda/textw/complete_text.py:42
msgid "Shutdown"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:80 pyanaconda/textw/complete_text.py:44
msgid "Please shutdown to use the installed system.\n"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:82 pyanaconda/textw/complete_text.py:46
msgid "Please reboot to use the installed system.\n"
msgstr ""

#: pyanaconda/iw/congrats_gui.py:87 pyanaconda/textw/complete_text.py:51
msgid ""
"Note that updates may be available to ensure the proper functioning of your "
"system and installation of these updates is recommended after the reboot."
msgstr ""

#: pyanaconda/iw/congrats_gui.py:91 pyanaconda/textw/complete_text.py:55
#, python-format
msgid ""
"Congratulations, your %s installation is complete.\n"
"\n"
"Please reboot to use the installed system.  Note that updates may be "
"available to ensure the proper functioning of your system and installation "
"of these updates is recommended after the reboot."
msgstr ""

#: pyanaconda/iw/examine_gui.py:39
msgid "Upgrade Examine"
msgstr ""

#: pyanaconda/iw/examine_gui.py:58
msgid "Fresh Installation"
msgstr ""

#: pyanaconda/iw/examine_gui.py:60
#, python-format
msgid ""
"Choose this option to install a fresh copy of %s on your system.  Existing "
"software and data may be overwritten depending on your configuration choices."
msgstr ""

#: pyanaconda/iw/examine_gui.py:65
msgid "Upgrade an Existing Installation"
msgstr ""

#: pyanaconda/iw/examine_gui.py:67
#, python-format
msgid ""
"Choose this option if you would like to upgrade your existing %s system.  "
"This option will preserve the existing data on your storage device(s)."
msgstr ""

#: pyanaconda/iw/examine_gui.py:117
msgid ""
"At least one existing installation has been detected on your system.  What "
"would you like to do?"
msgstr ""

#: pyanaconda/iw/examine_gui.py:136
#, python-format
msgid "<b>Which %s installation would you like to upgrade?</b>"
msgstr ""

#: pyanaconda/iw/examine_gui.py:150
msgid "Unknown Linux system"
msgstr ""

#: pyanaconda/iw/filter_gui.py:151
#, python-format
msgid "Selected devices: %s (%s MB) out of %s (%s MB)."
msgstr ""

#: pyanaconda/iw/filter_gui.py:401
msgid "Device Filter"
msgstr ""

#: pyanaconda/iw/filter_gui.py:494 pyanaconda/iw/filter_gui.py:524
#: pyanaconda/iw/filter_gui.py:535
msgid "Serial Number"
msgstr ""

#: pyanaconda/iw/filter_gui.py:495 pyanaconda/iw/filter_gui.py:503
#: pyanaconda/iw/filter_gui.py:514 pyanaconda/iw/filter_gui.py:525
#: pyanaconda/iw/filter_gui.py:540 pyanaconda/iw/osbootwidget.py:66
#: pyanaconda/iw/partition_gui.py:614
msgid "Device"
msgstr ""

#: pyanaconda/iw/filter_gui.py:513
msgid "Paths"
msgstr ""

#: pyanaconda/iw/filter_gui.py:537
msgid "Port"
msgstr ""

#: pyanaconda/iw/filter_gui.py:538
msgid "Target"
msgstr ""

#: pyanaconda/iw/filter_gui.py:539
msgid "LUN"
msgstr ""

#: pyanaconda/iw/filter_type.py:55
msgid "What type of devices will your installation involve?"
msgstr ""

#: pyanaconda/iw/filter_type.py:61
msgid "B_asic Storage Devices"
msgstr ""

#: pyanaconda/iw/filter_type.py:62
msgid ""
"Installs or upgrades to typical types of storage devices.  If you're not "
"sure which option is right for you, this is probably it."
msgstr ""

#: pyanaconda/iw/filter_type.py:66
msgid "_Specialized Storage Devices"
msgstr ""

#: pyanaconda/iw/filter_type.py:67
msgid ""
"Installs or upgrades to enterprise devices such as Storage Area Networks "
"(SANs). This option will allow you to add FCoE / iSCSI / zFCP disks and to "
"filter out devices the installer should ignore."
msgstr ""

#: pyanaconda/iw/language_gui.py:33 pyanaconda/textw/language_text.py:45
msgid "Language Selection"
msgstr ""

#: pyanaconda/iw/language_gui.py:81 loader/lang.c:360
#: pyanaconda/textw/language_text.py:46
msgid "What language would you like to use during the installation process?"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:128 pyanaconda/iw/lvm_dialog_gui.py:174
#: pyanaconda/iw/lvm_dialog_gui.py:188 pyanaconda/iw/lvm_dialog_gui.py:227
#: pyanaconda/iw/lvm_dialog_gui.py:304 pyanaconda/iw/lvm_dialog_gui.py:700
#: pyanaconda/iw/lvm_dialog_gui.py:721
msgid "Not enough space"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:129
msgid ""
"The physical extent size cannot be changed because otherwise the space "
"required by the currently defined logical volumes will be increased to more "
"than the available space."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:138
msgid "Confirm Physical Extent Change"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:139
msgid ""
"This change in the value of the physical extent will require the sizes of "
"the current logical volume requests to be rounded up in size to an integer "
"multiple of the physical extent.\n"
"\n"
"This change will take effect immediately."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:148 pyanaconda/iw/lvm_dialog_gui.py:210
msgid "C_ontinue"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:175
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is larger than the smallest physical volume "
"(%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:189
#, python-format
msgid ""
"The physical extent size cannot be changed because the value selected "
"(%(curpe)10.2f MB) is too large compared to the size of the smallest "
"physical volume (%(maxpvsize)10.2f MB) in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:203
msgid "Too small"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:204
msgid ""
"This change in the value of the physical extent will waste substantial space "
"on one or more of the physical volumes in the volume group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:228
#, python-format
msgid ""
"The physical extent size cannot be changed because the resulting maximum "
"logical volume size (%10.2f MB) is smaller than one or more of the currently "
"defined logical volumes."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:305
msgid ""
"You cannot remove this physical volume because otherwise the volume group "
"will be too small to hold the currently defined logical volumes."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:405
msgid "Make Logical Volume"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:407
#, python-format
msgid "Edit Logical Volume: %s"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:446 pyanaconda/iw/raid_dialog_gui.py:460
msgid "_File System Type:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:452
msgid "_Logical Volume Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:461
#: pyanaconda/iw/partition_dialog_gui.py:508
msgid "_Size (MB):"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:468
#, python-format
msgid "(Max size is %s MB)"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:473
#: pyanaconda/iw/partition_dialog_gui.py:560
#: pyanaconda/iw/partition_ui_helpers_gui.py:339
#: pyanaconda/iw/raid_dialog_gui.py:432
msgid "_Encrypt"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:482
#: pyanaconda/iw/partition_dialog_gui.py:490
#: pyanaconda/iw/raid_dialog_gui.py:470
msgid "Original File System Type:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:487
#: pyanaconda/iw/partition_dialog_gui.py:498
#: pyanaconda/iw/raid_dialog_gui.py:477
msgid "Original File System Label:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:492
msgid "Logical Volume Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:496
msgid "Size (MB):"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:523
#: pyanaconda/iw/partition_dialog_gui.py:441
#: pyanaconda/iw/raid_dialog_gui.py:451
msgid "_Mount Point:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:604
#: pyanaconda/iw/partition_dialog_gui.py:118
#: pyanaconda/iw/raid_dialog_gui.py:179
msgid "Mount Point Error"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:614
msgid "Illegal Logical Volume Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:629
msgid "Illegal logical volume name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:630
#, python-format
msgid "The logical volume name \"%s\" is already in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:671
#: pyanaconda/iw/partition_dialog_gui.py:134
#: pyanaconda/iw/raid_dialog_gui.py:194
msgid "Mount point in use"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:672
#: pyanaconda/iw/partition_dialog_gui.py:135
#: pyanaconda/iw/raid_dialog_gui.py:195
#, python-format
msgid "The mount point \"%s\" is in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:687
msgid "Illegal size"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:688
msgid "The requested size as entered is not a valid number greater than 0."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:701
#, python-format
msgid ""
"The current requested size (%(size)10.2f MB) is larger than the maximum "
"logical volume size (%(maxlv)10.2f MB). To increase this limit you can "
"create more Physical Volumes from unpartitioned disk space and add them to "
"this Volume Group."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:722
#, python-format
msgid ""
"The size entered for this logical volume (%(size)d MB) combined with the "
"size of the other logical volume(s) exceeds the size of the volume group "
"(%(tempvgsize)d MB). Please make the volume group larger or make the logical "
"volume smaller."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:840
msgid "No free slots"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:841
#, python-format
msgid "You cannot create more than %d logical volume per volume group."
msgid_plural "You cannot create more than %d logical volumes per volume group."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/lvm_dialog_gui.py:851
msgid "No free space"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:852
msgid ""
"There is no room left in the volume group to create new logical volumes. To "
"add a logical volume you must reduce the size of one or more of the "
"currently existing logical volumes"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:889
#, python-format
msgid "Are you sure you want to delete the logical volume \"%s\"?"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1004
msgid "Invalid Volume Group Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1013
msgid "Name in use"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1014
#, python-format
msgid "The volume group name \"%s\" is already in use. Please pick another."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1301
msgid "Not enough physical volumes"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1302
msgid ""
"At least one unused physical volume partition is needed to create an LVM "
"Volume Group.\n"
"\n"
"Create a partition or RAID array of type \"physical volume (LVM)\" and then "
"select the \"LVM\" option again."
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1313
msgid "Make LVM Volume Group"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1316
#, python-format
msgid "Edit LVM Volume Group: %s"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1318
msgid "Edit LVM Volume Group"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1334
msgid "_Volume Group Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1342
msgid "Volume Group Name:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1350
msgid "_Physical Extent:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1365
msgid "Physical Volumes to _Use:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1371
msgid "Used Space:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1389
msgid "Snapshot Total:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1409
msgid "Free Space:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1427
msgid "Total Space:"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1465
msgid "Logical Volume Name"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1468 pyanaconda/iw/partition_gui.py:617
#: pyanaconda/iw/upgrade_swap_gui.py:138 pyanaconda/textw/upgrade_text.py:124
msgid "Mount Point"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1471 pyanaconda/iw/partition_gui.py:616
msgid "Size (MB)"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1485 pyanaconda/iw/osbootwidget.py:96
msgid "_Add"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1488 pyanaconda/iw/osbootwidget.py:100
#: pyanaconda/iw/partition_gui.py:1814 pyanaconda/iw/partition_gui.py:1824
msgid "_Edit"
msgstr ""

#: pyanaconda/iw/lvm_dialog_gui.py:1503
msgid "_Logical Volumes"
msgstr ""

#: pyanaconda/iw/network_gui.py:81 pyanaconda/iw/network_gui.py:87
msgid "Error with Hostname"
msgstr ""

#: pyanaconda/iw/network_gui.py:82
msgid "You must enter a valid hostname for this computer."
msgstr ""

#: pyanaconda/iw/network_gui.py:88
#, python-format
msgid ""
"The hostname \"%(hostname)s\" is not valid for the following reason:\n"
"\n"
"%(herrors)s"
msgstr ""

#: pyanaconda/iw/network_gui.py:132
msgid "Select network interface"
msgstr ""

#: pyanaconda/iw/network_gui.py:139
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please configure a network interface."
msgstr ""

#: pyanaconda/iw/network_gui.py:209
msgid "Select APs"
msgstr ""

#: pyanaconda/iw/network_gui.py:216
msgid "Select APs for wireless devices"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:50
msgid "Boot loader operating system list"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:66
msgid "Default"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:66 pyanaconda/iw/partition_gui.py:615
msgid "Label"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:130
msgid "Image"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:137
msgid ""
"Enter a label for the boot loader menu to display. The device (or hard drive "
"and partition number) is the device from which it boots."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:145
msgid "_Label"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:153
msgid "_Device"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:183
msgid "Default Boot _Target"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:212
msgid "You must select a device."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:218
msgid "You must specify a label for the entry"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:227
msgid "Boot label contains illegal characters"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:251
msgid "Duplicate Label"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:252
msgid "This label is already in use for another boot entry."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:263
msgid "Duplicate Device"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:264
msgid "This device is already being used for another boot entry."
msgstr ""

#: pyanaconda/iw/osbootwidget.py:326
msgid "Cannot Delete"
msgstr ""

#: pyanaconda/iw/osbootwidget.py:327
#, python-format
msgid ""
"This boot target cannot be deleted because it is for the %s system you are "
"about to install."
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:58
msgid "Additional Size Options"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:63
msgid "_Fixed size"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:65
msgid "Fill all space _up to (MB):"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:72
msgid "Fill to maximum _allowable size"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:193
msgid "Error With Request"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:411
msgid "Add Partition"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:413
#, python-format
msgid "Edit Partition: %s"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:450
msgid "File System _Type:"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:466
msgid "Allowable _Drives:"
msgstr ""

#: pyanaconda/iw/partition_dialog_gui.py:547
msgid "Force to be a _primary partition"
msgstr ""

#: pyanaconda/iw/partition_gui.py:360
#, python-format
msgid "Drive %(drive)s (%(size)-0.f MB) (Model: %(model)s)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:400 pyanaconda/iw/partition_gui.py:449
#: pyanaconda/iw/partition_gui.py:516 pyanaconda/iw/partition_gui.py:1001
#: pyanaconda/iw/partition_gui.py:1073
msgid "Free"
msgstr ""

#: pyanaconda/iw/partition_gui.py:481
#, python-format
msgid "LVM Volume Group %s (%-0.f MB)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:547
#, python-format
msgid "MD RAID ARRAY %s (%-0.f MB)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:618
msgid "Type"
msgstr ""

#: pyanaconda/iw/partition_gui.py:619 pyanaconda/storage/__init__.py:1851
msgid "Format"
msgstr ""

#: pyanaconda/iw/partition_gui.py:656
msgid ""
"Mount Point/\n"
"RAID/Volume"
msgstr ""

#: pyanaconda/iw/partition_gui.py:658
msgid ""
"Size\n"
"(MB)"
msgstr ""

#: pyanaconda/iw/partition_gui.py:763
msgid "Partitioning"
msgstr ""

#: pyanaconda/iw/partition_gui.py:853
msgid ""
"The partitioning scheme you requested caused the following critical errors."
msgstr ""

#: pyanaconda/iw/partition_gui.py:855
#, python-format
msgid ""
"You must correct these errors before you continue your installation of %s."
msgstr ""

#: pyanaconda/iw/partition_gui.py:861
msgid "Partitioning Errors"
msgstr ""

#: pyanaconda/iw/partition_gui.py:868
msgid "The partitioning scheme you requested generated the following warnings."
msgstr ""

#: pyanaconda/iw/partition_gui.py:870
msgid "Would you like to continue with your requested partitioning scheme?"
msgstr ""

#: pyanaconda/iw/partition_gui.py:875
msgid "Partitioning Warnings"
msgstr ""

#: pyanaconda/iw/partition_gui.py:884
msgid ""
"The following pre-existing devices have been selected to be formatted, "
"destroying all data."
msgstr ""

#: pyanaconda/iw/partition_gui.py:896
msgid "Format Warnings"
msgstr ""

#: pyanaconda/iw/partition_gui.py:989
msgid "LVM Volume Groups"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1011
msgid "RAID Devices"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1027 loader/hdinstall.c:168
msgid "Hard Drives"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1077
msgid "Extended"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1367
msgid "Cannot perform any creation action"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1368
msgid ""
"Note that the creation action requires one of the following:\n"
"\n"
"* Free space in one of the Hard Drives.\n"
"* At least two free Software RAID partitions.\n"
"* At least one free physical volume (LVM) partition.\n"
"* At least one Volume Group with free space."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1471
#, python-format
msgid ""
"Software RAID allows you to combine several disks into a larger RAID "
"device.  A RAID device can be configured to provide additional speed and "
"reliability compared to using an individual drive.  For more information on "
"using RAID devices please consult the %s documentation.\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1477
msgid ""
"To use RAID you must first create at least two partitions of type 'software "
"RAID'.  Then you can create a RAID device that can be formatted and "
"mounted.\n"
"\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1481
#, python-format
msgid "You currently have %d software RAID partition free to use."
msgid_plural "You currently have %d software RAID partitions free to use."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/partition_gui.py:1485
msgid "About RAID"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1491
#, python-format
msgid ""
"Logical Volume Manager (LVM) is a 3 level construct. The first level is made "
"up of disks or partitions formatted with LVM metadata called Physical "
"Volumes (PV).  A Volume Group (VG) sits on top of one or more PVs. The VG, "
"in turn, is the base to create one or more Logical Volumes (LV).  Note that "
"a VG can be an aggregate of PVs from multiple physical disks.  For more "
"information on using LVM please consult the %s documentation\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1499
msgid ""
"To create a PV you need a partition with free space.  To create a VG you "
"need a PV that is not part of any existing VG.  To create an LV you need a "
"VG with free space.\n"
"\n"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1503
#, python-format
msgid "You currently have %d available PV free to use.\n"
msgid_plural "You currently have %d available PVs free to use.\n"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/iw/partition_gui.py:1507
msgid "You currently have free space to create PVs."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1510
msgid "About LVM"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1588 pyanaconda/storage/partitioning.py:206
#: pyanaconda/storage/partitioning.py:248
msgid "Error Partitioning"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1589
#, python-format
msgid "Could not allocate requested partitions: %s."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1598
#, python-format
msgid "Warning: %s."
msgstr ""

#: pyanaconda/iw/partition_gui.py:1629
msgid "Unable To Edit"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1630
#, python-format
msgid ""
"You cannot edit this device:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1813
msgid "_Create"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1816
msgid "Re_set"
msgstr ""

#: pyanaconda/iw/partition_gui.py:1839
msgid "Please Select A Device"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:110
#: pyanaconda/iw/partition_ui_helpers_gui.py:132
#: pyanaconda/iw/partition_ui_helpers_gui.py:134
msgid "<Not Applicable>"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:320
msgid "_Format as:"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:342
msgid "Mi_grate filesystem to:"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:372
msgid "_Resize"
msgstr ""

#: pyanaconda/iw/partition_ui_helpers_gui.py:427
#, python-format
msgid ""
"Partitions of type '%s' must be constrained to a single drive.  To do this, "
"select the drive in the 'Allowable Drives' checklist."
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:399
msgid ""
"At least two unused software RAID partitions are needed to create a RAID "
"device.\n"
"\n"
"First create at least two partitions of type \"software RAID\", and then "
"select the \"RAID\" option again."
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:413
msgid "Make RAID Device"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:416
#, python-format
msgid "Edit RAID Device: %s"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:418
msgid "Edit RAID Device"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:485
msgid "RAID _Device:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:503
msgid "RAID _Level:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:545
msgid "_RAID Members:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:562
msgid "Number of _spares:"
msgstr ""

#: pyanaconda/iw/raid_dialog_gui.py:572
msgid "_Format partition?"
msgstr ""

#: pyanaconda/iw/task_gui.py:53
#, python-format
msgid ""
"Unable to read package metadata from repository.  This may be due to a "
"missing repodata directory.  Please ensure that your repository has been "
"correctly generated.\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/iw/task_gui.py:127
msgid "Edit Repository"
msgstr ""

#: pyanaconda/iw/task_gui.py:151
#, python-format
msgid ""
"The repository %s has already been added.  Please choose a different "
"repository name and URL."
msgstr ""

#: pyanaconda/iw/task_gui.py:237
msgid "Invalid Proxy URL"
msgstr ""

#: pyanaconda/iw/task_gui.py:238
msgid "You must provide an HTTP, HTTPS, or FTP URL to a proxy."
msgstr ""

#: pyanaconda/iw/task_gui.py:250 pyanaconda/iw/task_gui.py:432
msgid "Invalid Repository URL"
msgstr ""

#: pyanaconda/iw/task_gui.py:251 pyanaconda/iw/task_gui.py:433
msgid "You must provide an HTTP, HTTPS, or FTP URL to a repository."
msgstr ""

#: pyanaconda/iw/task_gui.py:272 pyanaconda/iw/task_gui.py:442
msgid "No Media Found"
msgstr ""

#: pyanaconda/iw/task_gui.py:273 pyanaconda/iw/task_gui.py:443
msgid ""
"No installation media was found. Please insert a disc into your drive and "
"try again."
msgstr ""

#: pyanaconda/iw/task_gui.py:303 pyanaconda/iw/task_gui.py:465
msgid "Please enter an NFS server and path."
msgstr ""

#: pyanaconda/iw/task_gui.py:321
#, python-format
msgid ""
"The following error occurred while setting up the repository:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/iw/task_gui.py:344
msgid "Invalid Repository Name"
msgstr ""

#: pyanaconda/iw/task_gui.py:345
msgid "You must provide a repository name."
msgstr ""

#: pyanaconda/iw/task_gui.py:497 data/ui/addrepo.glade.h:3
msgid "Add Repository"
msgstr ""

#: pyanaconda/iw/task_gui.py:502
msgid "No Software Repos Enabled"
msgstr ""

#: pyanaconda/iw/task_gui.py:503
msgid ""
"You must have at least one software repository enabled to continue "
"installation."
msgstr ""

#: pyanaconda/iw/timezone_gui.py:53 pyanaconda/textw/timezone_text.py:68
msgid "Time Zone Selection"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:37
#: pyanaconda/textw/upgrade_bootloader_text.py:129
msgid "Upgrade Boot Loader Configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:120
msgid "_Update boot loader configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:121
msgid "This will update your current boot loader."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:125
#: pyanaconda/textw/upgrade_bootloader_text.py:103
msgid ""
"Due to system changes, your boot loader configuration can not be "
"automatically updated."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:128
#: pyanaconda/textw/upgrade_bootloader_text.py:107
msgid ""
"The installer is unable to detect the boot loader currently in use on your "
"system."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:135
#: pyanaconda/textw/upgrade_bootloader_text.py:116
#, python-format
msgid ""
"The installer has detected the %(type)s boot loader currently installed on "
"%(bootDev)s."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:139
msgid "This is the recommended option."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:144
msgid "_Create new boot loader configuration"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:146
msgid ""
"This option creates a new boot loader configuration.  If you wish to switch "
"boot loaders, you should choose this."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:153
msgid "_Skip boot loader updating"
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:154
msgid ""
"This option makes no changes to boot loader configuration.  If you are using "
"a third party boot loader, you should choose this."
msgstr ""

#: pyanaconda/iw/upgrade_bootloader_gui.py:165
msgid "What would you like to do?"
msgstr ""

#: pyanaconda/iw/upgrade_migratefs_gui.py:38
#: pyanaconda/textw/upgrade_text.py:42
msgid "Migrate File Systems"
msgstr ""

#: pyanaconda/iw/upgrade_migratefs_gui.py:69
#: pyanaconda/textw/upgrade_text.py:44
#, python-format
msgid ""
"This release of %(productName)s supports an updated file system, which has "
"several benefits over the file system traditionally shipped in "
"%(productName)s.  This installation program can migrate formatted partitions "
"without data loss.\n"
"\n"
"Which of these partitions would you like to migrate?"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:35
msgid "Upgrade Swap Partition"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:92 pyanaconda/textw/upgrade_text.py:107
#, python-format
msgid ""
"Recent kernels (2.4 or newer) need significantly more swap than older "
"kernels, up to twice the amount of RAM on the system.  You currently have "
"%dMB of swap configured, but you may create additional swap space on one of "
"your file systems now."
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:99
#, python-format
msgid ""
"\n"
"\n"
"The installer has detected %s MB of RAM.\n"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:111
msgid "I _want to create a swap file"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:120
msgid "Select the _partition to put the swap file on:"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:138 pyanaconda/textw/upgrade_text.py:125
msgid "Partition"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:138
msgid "Free Space (MB)"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:156
#, python-format
msgid ""
"A minimum swap file size of %d MB is recommended.  Please enter a size for "
"the swap file:"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:171
msgid "Swap file _size (MB):"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:181
msgid "I _don't want to create a swap file"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:191
msgid ""
"A swap file is strongly recommended. Failure to create one could cause the "
"installer to abort abnormally.  Are you sure you wish to continue?"
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:199 pyanaconda/textw/upgrade_text.py:194
msgid "The swap file must be between 1 and 2000 MB in size."
msgstr ""

#: pyanaconda/iw/upgrade_swap_gui.py:206 pyanaconda/textw/upgrade_text.py:189
msgid ""
"There is not enough space on the device you selected for the swap partition."
msgstr ""

#: pyanaconda/iw/zipl_gui.py:37
msgid "z/IPL Boot Loader Configuration"
msgstr ""

#: pyanaconda/iw/zipl_gui.py:61
msgid "The z/IPL boot loader will be installed on your system."
msgstr ""

#: pyanaconda/iw/zipl_gui.py:63
msgid ""
"The z/IPL Boot Loader will now be installed on your system.\n"
"\n"
"The root partition will be the one you selected previously in the partition "
"setup.\n"
"\n"
"The kernel used to start the machine will be the one to be installed by "
"default.\n"
"\n"
"If you wish to make changes later after the installation feel free to change "
"the /etc/zipl.conf configuration file.\n"
"\n"
"You can now enter any additional kernel parameters which your machine or "
"your setup may require."
msgstr ""

#: pyanaconda/iw/zipl_gui.py:90 pyanaconda/textw/zipl_text.py:72
msgid "Kernel Parameters"
msgstr ""

#: pyanaconda/iw/zipl_gui.py:93 pyanaconda/iw/zipl_gui.py:96
msgid "Chandev Parameters"
msgstr ""

#: loader/net.h:27 pyanaconda/textw/netconfig_text.py:164
msgid "Dynamic IP configuration (DHCP)"
msgstr ""

#: loader/net.h:28
msgid "Manual configuration"
msgstr ""

#: loader/net.h:30
msgid "Dynamic IP configuration (DHCPv6)"
msgstr ""

#: loader/net.h:31 pyanaconda/textw/netconfig_text.py:201
msgid "Automatic neighbor discovery"
msgstr ""

#: loader/cdinstall.c:187
#, c-format
msgid ""
"The %s disc was not found in any of your drives. Please insert the %s disc "
"and press %s to retry."
msgstr ""

#: loader/cdinstall.c:210
msgid "Disc Found"
msgstr ""

#: loader/cdinstall.c:211
#, c-format
msgid ""
"To begin testing the media before installation press %s.\n"
"\n"
"Choose %s to skip the media test and start the installation."
msgstr ""

#: loader/cdinstall.c:273
msgid "Scanning"
msgstr ""

#: loader/cdinstall.c:273
#, c-format
msgid "Looking for installation media on CD device %s\n"
msgstr ""

#: loader/cdinstall.c:275
#, c-format
msgid "Looking for installation media on CD device %s"
msgstr ""

#: loader/cdinstall.c:334
#, c-format
msgid ""
"The %s disc was not found in any of your CDROM drives. Please insert the %s "
"disc and press %s to retry."
msgstr ""

#: loader/cdinstall.c:339
msgid "Disc Not Found"
msgstr ""

#: loader/cdinstall.c:340 loader/driverdisk.c:393 loader/driverdisk.c:425
#: loader/driverdisk.c:506 loader/driverselect.c:79 loader/driverselect.c:145
#: loader/driverselect.c:171 loader/hdinstall.c:168 loader/hdinstall.c:222
#: loader/kbd.c:118 loader/loader.c:499 loader/loader.c:516
#: loader/loader.c:1389 loader/loader.c:1429 loader/net.c:624
#: loader/net.c:1004 loader/net.c:1813 loader/net.c:1832
#: loader/nfsinstall.c:93 loader/urls.c:264 pyanaconda/storage/__init__.py:122
#: pyanaconda/storage/__init__.py:168 pyanaconda/storage/devicetree.py:95
#: pyanaconda/textw/constants_text.py:52
msgid "Back"
msgstr ""

#: loader/cdinstall.c:417
msgid "Cannot find kickstart file on CDROM."
msgstr ""

#: loader/copy.c:51 loader/method.c:195
#, c-format
msgid "Failed to read directory %s: %m"
msgstr ""

#: loader/driverdisk.c:256
msgid "Loading"
msgstr ""

#: loader/driverdisk.c:256
msgid "Reading driver disk"
msgstr ""

#: loader/driverdisk.c:387 loader/driverdisk.c:420
msgid "Driver Disk Source"
msgstr ""

#: loader/driverdisk.c:388
msgid ""
"You have multiple devices which could serve as sources for a driver disk.  "
"Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:421
msgid ""
"There are multiple partitions on this device which could contain the driver "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/driverdisk.c:456
msgid "Failed to mount partition."
msgstr ""

#: loader/driverdisk.c:461
msgid "Select driver disk image"
msgstr ""

#: loader/driverdisk.c:462
msgid "Select the file which is your driver disk image."
msgstr ""

#: loader/driverdisk.c:491
msgid "Failed to load driver disk from file."
msgstr ""

#: loader/driverdisk.c:503
#, c-format
msgid "Insert your driver disk into /dev/%s and press \"OK\" to continue."
msgstr ""

#: loader/driverdisk.c:506
msgid "Insert Driver Disk"
msgstr ""

#: loader/driverdisk.c:519
msgid "Failed to mount driver disk."
msgstr ""

#: loader/driverdisk.c:527
#, c-format
msgid "Driver disk is invalid for this release of %s."
msgstr ""

#: loader/driverdisk.c:598
msgid "Manually choose"
msgstr ""

#: loader/driverdisk.c:599
msgid "Load another disk"
msgstr ""

#: loader/driverdisk.c:600
msgid ""
"No devices of the appropriate type were found on this driver disk.  Would "
"you like to manually select the driver, continue anyway, or load another "
"driver disk?"
msgstr ""

#: loader/driverdisk.c:638
msgid "Driver disk"
msgstr ""

#: loader/driverdisk.c:639
msgid "Do you have a driver disk?"
msgstr ""

#: loader/driverdisk.c:648
msgid "More Driver Disks?"
msgstr ""

#: loader/driverdisk.c:649
msgid "Do you wish to load any more driver disks?"
msgstr ""

#: loader/driverdisk.c:698 loader/kickstart.c:176 loader/kickstart.c:922
msgid "Kickstart Error"
msgstr ""

#: loader/driverdisk.c:699
#, c-format
msgid "Unknown driver disk kickstart source: %s"
msgstr ""

#: loader/driverselect.c:68
#, c-format
msgid ""
"Please enter any parameters which you wish to pass to the %s module "
"separated by spaces.  If you don't know what parameters to supply, skip this "
"screen by pressing the \"OK\" button."
msgstr ""

#: loader/driverselect.c:89
msgid "Enter Module Parameters"
msgstr ""

#: loader/driverselect.c:144
msgid "No drivers found"
msgstr ""

#: loader/driverselect.c:144
msgid "Load driver disk"
msgstr ""

#: loader/driverselect.c:145
msgid ""
"No drivers were found to manually insert.  Would you like to use a driver "
"disk?"
msgstr ""

#: loader/driverselect.c:163
msgid ""
"Please select the driver below which you wish to load.  If it does not "
"appear and you have a driver disk, press F2."
msgstr ""

#: loader/driverselect.c:172
msgid "Specify optional module arguments"
msgstr ""

#: loader/driverselect.c:197
msgid "Select Device Driver to Load"
msgstr ""

#: loader/hdinstall.c:169
msgid ""
"You don't seem to have any hard drives on your system! Would you like to "
"configure additional devices?"
msgstr ""

#: loader/hdinstall.c:182
#, c-format
msgid ""
"What partition and directory on that partition holds the installation image "
"for %s?  If you don't see the disk drive you're using listed here, press F2 "
"to configure additional devices."
msgstr ""

#: loader/hdinstall.c:206
msgid "Directory holding image:"
msgstr ""

#: loader/hdinstall.c:234
msgid "Select Partition"
msgstr ""

#: loader/hdinstall.c:283 loader/hdinstall.c:301 loader/hdinstall.c:310
msgid "That directory does not contain installation media."
msgstr ""

#: loader/hdinstall.c:345 loader/hdinstall.c:399
msgid "Cannot find kickstart file on hard drive."
msgstr ""

#: loader/hdinstall.c:384
#, c-format
msgid "Cannot find hard drive for BIOS disk %s"
msgstr ""

#: loader/kbd.c:116
msgid "Keyboard Type"
msgstr ""

#: loader/kbd.c:117
msgid "What type of keyboard do you have?"
msgstr ""

#: loader/kickstart.c:309
msgid "Cannot find ks.cfg on removable media."
msgstr ""

#: loader/kickstart.c:378
msgid ""
"Unable to download the kickstart file.  Please modify the kickstart "
"parameter below or press Cancel to proceed as an interactive installation."
msgstr ""

#: loader/kickstart.c:387
msgid "Error downloading kickstart file"
msgstr ""

#: loader/kickstart.c:923
#, c-format
msgid "Unknown Url method %s"
msgstr ""

#: loader/lang.c:62 loader/loader.c:228
#, c-format
msgid "Welcome to %s for %s"
msgstr ""

#: loader/lang.c:63
#, c-format
msgid "Welcome to %s for %s - Rescue Mode"
msgstr ""

#: loader/lang.c:64 loader/loader.c:252
msgid ""
"  <Tab>/<Alt-Tab> between elements  | <Space> selects | <F12> next screen "
msgstr ""

#: loader/lang.c:359
msgid "Choose a Language"
msgstr ""

#: loader/loader.c:132
msgid "Local CD/DVD"
msgstr ""

#: loader/loader.c:133
msgid "Hard drive"
msgstr ""

#: loader/loader.c:134
msgid "NFS directory"
msgstr ""

#: loader/loader.c:453 loader/loader.c:494
msgid "Update Disk Source"
msgstr ""

#: loader/loader.c:454
msgid ""
"You have multiple devices which could serve as sources for an update disk.  "
"Which would you like to use?"
msgstr ""

#: loader/loader.c:495
msgid ""
"There are multiple partitions on this device which could contain the update "
"disk image.  Which would you like to use?"
msgstr ""

#: loader/loader.c:513
#, c-format
msgid "Insert your updates disk into %s and press \"OK\" to continue."
msgstr ""

#: loader/loader.c:516
msgid "Updates Disk"
msgstr ""

#: loader/loader.c:534
msgid "Failed to mount updates disk"
msgstr ""

#: loader/loader.c:539
msgid "Updates"
msgstr ""

#: loader/loader.c:539
msgid "Reading anaconda updates"
msgstr ""

#: loader/loader.c:575
msgid ""
"Unable to download the updates image.  Please modify the updates location "
"below or press Cancel to proceed without updates.."
msgstr ""

#: loader/loader.c:584
msgid "Error downloading updates image"
msgstr ""

#: loader/loader.c:1198
#, c-format
msgid "%s requires %d MB of memory, to install, but you only have %d MB."
msgstr ""

#: loader/loader.c:1199
#, c-format
msgid ""
"%s requires %d MB of memory to install using this installation method, but "
"you only have %d MB on this machine."
msgstr ""

#: loader/loader.c:1244
msgid "Unsupported Hardware Detected"
msgstr ""

#: loader/loader.c:1245
msgid ""
"This hardware (or a combination thereof) is not supported by Red Hat.  For "
"more information on supported hardware, please refer to http://www.redhat."
"com/hardware."
msgstr ""

#: loader/loader.c:1381
msgid "Rescue Method"
msgstr ""

#: loader/loader.c:1382
msgid "Installation Method"
msgstr ""

#: loader/loader.c:1384
msgid "What type of media contains the rescue image?"
msgstr ""

#: loader/loader.c:1386
msgid "What type of media contains the installation image?"
msgstr ""

#: loader/loader.c:1428
msgid "No driver found"
msgstr ""

#: loader/loader.c:1428
msgid "Select driver"
msgstr ""

#: loader/loader.c:1429
msgid "Use a driver disk"
msgstr ""

#: loader/loader.c:1430
msgid ""
"Unable to find any devices of the type needed for this installation type.  "
"Would you like to manually select your driver or use a driver disk?"
msgstr ""

#: loader/loader.c:1641
msgid "The following devices have been found on your system."
msgstr ""

#: loader/loader.c:1643
msgid ""
"No device drivers have been loaded for your system.  Would you like to load "
"any now?"
msgstr ""

#: loader/loader.c:1647
msgid "Devices"
msgstr ""

#: loader/loader.c:1648
msgid "Done"
msgstr ""

#: loader/loader.c:1649
msgid "Add Device"
msgstr ""

#: loader/loader.c:1931
#, c-format
msgid "loader has already been run.  Starting shell.\n"
msgstr ""

#: loader/loader.c:2075
#, c-format
msgid "Driver disc was detected in %s. Do you want to use it?."
msgstr ""

#: loader/loader.c:2078
msgid "Driver disc detected"
msgstr ""

#: loader/loader.c:2078
msgid "Use it"
msgstr ""

#: loader/loader.c:2078
msgid "Skip it"
msgstr ""

#: loader/loader.c:2392
#, c-format
msgid "Running anaconda %s, the %s rescue mode - please wait.\n"
msgstr ""

#: loader/loader.c:2394
#, c-format
msgid "Running anaconda %s, the %s system installer - please wait.\n"
msgstr ""

#: loader/mediacheck.c:47
#, c-format
msgid "Unable to find install image %s"
msgstr ""

#: loader/mediacheck.c:53
#, c-format
msgid "Checking \"%s\"."
msgstr ""

#: loader/mediacheck.c:55
#, c-format
msgid "Checking media."
msgstr ""

#: loader/mediacheck.c:61
msgid "Media Check"
msgstr ""

#: loader/mediacheck.c:87
msgid ""
"Unable to find the checksum in the image.  This probably means the disc was "
"created without adding the checksum."
msgstr ""

#: loader/mediacheck.c:94
msgid "Unable to open the image."
msgstr ""

#: loader/mediacheck.c:98
msgid ""
"The image which was just tested has errors. This could be due to a corrupt "
"download or a bad disc.  If applicable, please clean the disc and try "
"again.  If this test continues to fail you should not continue the install."
msgstr ""

#: loader/mediacheck.c:106
msgid "Success"
msgstr ""

#: loader/mediacheck.c:107
msgid ""
"The image which was just tested was successfully verified.  It should be OK "
"to install from this media.  Note that not all media/drive errors can be "
"detected by the media check."
msgstr ""

#: loader/method.c:238
#, c-format
msgid ""
"Would you like to perform a checksum test of the ISO image:\n"
"\n"
"   %s?"
msgstr ""

#: loader/method.c:241
msgid "Checksum Test"
msgstr ""

#: loader/method.c:241
msgid "Test"
msgstr ""

#: loader/net.c:109
msgid "Invalid Prefix"
msgstr ""

#: loader/net.c:110
msgid ""
"Prefix must be between 1 and 32 for IPv4 networks or between 1 and 128 for "
"IPv6 networks"
msgstr ""

#: loader/net.c:505 loader/net.c:513 loader/net.c:571
#, c-format
msgid "There was an error configuring your network interface."
msgstr ""

#: loader/net.c:507
#, c-format
msgid ""
"\n"
"This cannot be corrected in cmdline mode.\n"
"Halting.\n"
msgstr ""

#: loader/net.c:512 loader/net.c:570 pyanaconda/textw/netconfig_text.py:403
msgid "Network Error"
msgstr ""

#: loader/net.c:602
msgid "Enable IPv4 support"
msgstr ""

#: loader/net.c:616
msgid "Enable IPv6 support"
msgstr ""

#: loader/net.c:656
msgid "Configure TCP/IP"
msgstr ""

#: loader/net.c:715
msgid "Missing Protocol"
msgstr ""

#: loader/net.c:716
msgid "You must select at least one protocol (IPv4 or IPv6)."
msgstr ""

#: loader/net.c:723
msgid "IPv4 Needed for NFS"
msgstr ""

#: loader/net.c:724
msgid "NFS installation method requires IPv4 support."
msgstr ""

#: loader/net.c:830
msgid "IPv4 address:"
msgstr ""

#: loader/net.c:842 loader/net.c:909
msgid "/"
msgstr ""

#: loader/net.c:897
msgid "IPv6 address:"
msgstr ""

#: loader/net.c:963 pyanaconda/textw/netconfig_text.py:184
#: pyanaconda/textw/netconfig_text.py:224
msgid "Gateway:"
msgstr ""

#: loader/net.c:971
msgid "Name Server:"
msgstr ""

#: loader/net.c:1010
msgid ""
"Enter the IPv4 and/or the IPv6 address and prefix (address / prefix).  For "
"IPv4, the dotted-quad netmask or the CIDR-style prefix are acceptable. The "
"gateway and name server fields must be valid IPv4 or IPv6 addresses."
msgstr ""

#: loader/net.c:1027
msgid "Manual TCP/IP Configuration"
msgstr ""

#: loader/net.c:1151 loader/net.c:1159
msgid "Missing Information"
msgstr ""

#: loader/net.c:1152
msgid ""
"You must enter both a valid IPv4 address and a network mask or CIDR prefix."
msgstr ""

#: loader/net.c:1160
msgid "You must enter both a valid IPv6 address and a CIDR prefix."
msgstr ""

#: loader/net.c:1613
msgid "Seconds:"
msgstr ""

#: loader/net.c:1808
msgid "Networking Device"
msgstr ""

#: loader/net.c:1809
msgid ""
"You have multiple network devices on this system. Which would you like to "
"install through?"
msgstr ""

#: loader/net.c:1813
msgid "Identify"
msgstr ""

#: loader/net.c:1822
msgid "You can identify the physical port for"
msgstr ""

#: loader/net.c:1824
msgid ""
"by flashing the LED lights for a number of seconds.  Enter a number between "
"1 and 30 to set the duration to flash the LED port lights."
msgstr ""

#: loader/net.c:1831
msgid "Identify NIC"
msgstr ""

#: loader/net.c:1844
msgid "Invalid Duration"
msgstr ""

#: loader/net.c:1845
msgid "You must enter the number of seconds as an integer between 1 and 30."
msgstr ""

#: loader/net.c:1857
#, c-format
msgid "Flashing %s port lights for %d seconds."
msgstr ""

#: loader/net.c:2029 loader/net.c:2033
#, c-format
msgid "Waiting for NetworkManager to configure %s.\n"
msgstr ""

#: loader/nfsinstall.c:70
msgid "NFS server name:"
msgstr ""

#: loader/nfsinstall.c:74
#, c-format
msgid "%s directory:"
msgstr ""

#: loader/nfsinstall.c:78
msgid "NFS mount options (optional):"
msgstr ""

#: loader/nfsinstall.c:84
#, c-format
msgid ""
"Please enter the server and path to your %s installation image and "
"optionally additional NFS mount options."
msgstr ""

#: loader/nfsinstall.c:92
msgid "NFS Setup"
msgstr ""

#: loader/nfsinstall.c:256 loader/urlinstall.c:214
msgid "The URL provided does not contain installation media."
msgstr ""

#: loader/urlinstall.c:140
#, c-format
msgid "Unable to retrieve %s."
msgstr ""

#: loader/urls.c:173 loader/urls.c:181
msgid "Retrieving"
msgstr ""

#: loader/urls.c:267
#, c-format
msgid ""
"Please enter the URL containing the %s installation image on your server."
msgstr ""

#: loader/urls.c:285
msgid "Enable HTTP proxy"
msgstr ""

#: loader/urls.c:299
msgid "Proxy URL"
msgstr ""

#: loader/urls.c:304
msgid "Username"
msgstr ""

#: loader/urls.c:309
msgid "Password"
msgstr ""

#: loader/urls.c:328
msgid "URL Setup"
msgstr ""

#: loader/urls.c:336
msgid "You must enter a URL."
msgstr ""

#: loader/urls.c:344
msgid "URL must be either an ftp or http URL"
msgstr ""

#: loader/urls.c:351
msgid "Proxy must be either an ftp or http URL"
msgstr ""

#: pyanaconda/storage/__init__.py:109
msgid "Unknown Device"
msgstr ""

#: pyanaconda/storage/__init__.py:110
#, python-format
msgid ""
"The installation source given by device %s could not be found.  Please check "
"your parameters and try again."
msgstr ""

#: pyanaconda/storage/__init__.py:119
msgid "No disks found"
msgstr ""

#: pyanaconda/storage/__init__.py:120
msgid "No usable disks have been found."
msgstr ""

#: pyanaconda/storage/__init__.py:131
msgid "Installation cannot continue."
msgstr ""

#: pyanaconda/storage/__init__.py:132
msgid ""
"The storage configuration you have chosen has already been activated. You "
"can no longer return to the disk editing screen. Would you like to continue "
"with the installation process?"
msgstr ""

#: pyanaconda/storage/__init__.py:160
msgid "Encrypt device?"
msgstr ""

#: pyanaconda/storage/__init__.py:161
msgid ""
"You specified block device encryption should be enabled, but you have not "
"supplied a passphrase. If you do not go back and provide a passphrase, block "
"device encryption will be disabled."
msgstr ""

#: pyanaconda/storage/__init__.py:196
msgid ""
"\n"
"\n"
"<b>WARNING:</b>\n"
"You are using a GPT bootdisk on a non-EFI system. This may not work, "
"depending on your BIOS's support for booting from GPT disks."
msgstr ""

#: pyanaconda/storage/__init__.py:204 pyanaconda/storage/devicetree.py:104
msgid "Confirm"
msgstr ""

#: pyanaconda/storage/__init__.py:205
#, python-format
msgid ""
"The partitioning options you have selected will now be written to disk.  Any "
"data on deleted or reformatted partitions will be lost.%s"
msgstr ""

#: pyanaconda/storage/__init__.py:211
msgid "Go _Back"
msgstr ""

#: pyanaconda/storage/__init__.py:212
msgid "_Write Changes to Disk"
msgstr ""

#: pyanaconda/storage/__init__.py:231
msgid "Running..."
msgstr ""

#: pyanaconda/storage/__init__.py:232
msgid "Storing encryption keys"
msgstr ""

#: pyanaconda/storage/__init__.py:248
#, python-format
msgid "Error storing an encryption key: %s\n"
msgstr ""

#: pyanaconda/storage/__init__.py:416
msgid "Examining Devices"
msgstr ""

#: pyanaconda/storage/__init__.py:417
msgid "Examining storage devices"
msgstr ""

#: pyanaconda/storage/__init__.py:701
msgid "This partition is holding the data for the hard drive install."
msgstr ""

#: pyanaconda/storage/__init__.py:706
msgid "You cannot delete a partition of a LDL formatted DASD."
msgstr ""

#: pyanaconda/storage/__init__.py:712
#, python-format
msgid "This device is part of the RAID device %s."
msgstr ""

#: pyanaconda/storage/__init__.py:715
msgid "This device is part of a RAID device."
msgstr ""

#: pyanaconda/storage/__init__.py:720
#, python-format
msgid "This device is part of the LVM volume group '%s'."
msgstr ""

#: pyanaconda/storage/__init__.py:723
msgid "This device is part of a LVM volume group."
msgstr ""

#: pyanaconda/storage/__init__.py:739
msgid ""
"This device is an extended partition which contains logical partitions that "
"cannot be deleted:\n"
"\n"
msgstr ""

#: pyanaconda/storage/__init__.py:896
msgid "You must create a new filesystem on the root device."
msgstr ""

#: pyanaconda/storage/__init__.py:1028
#, python-format
msgid ""
"You have not defined a root partition (/), which is required for "
"installation of %s to continue."
msgstr ""

#: pyanaconda/storage/__init__.py:1033
#, python-format
msgid ""
"Your root partition is less than 250 megabytes which is usually too small to "
"install %s."
msgstr ""

#: pyanaconda/storage/__init__.py:1043
#, python-format
msgid ""
"Your / partition is less than %(min)s MB which is lower than recommended for "
"a normal %(productName)s%(live)s install."
msgstr ""

#: pyanaconda/storage/__init__.py:1054
#, python-format
msgid ""
"Your / partition does not match the the live image you are installing from.  "
"It must be formatted as %s."
msgstr ""

#: pyanaconda/storage/__init__.py:1061
#, python-format
msgid ""
"Your %(mount)s partition is less than %(size)s megabytes which is lower than "
"recommended for a normal %(productName)s install."
msgstr ""

#: pyanaconda/storage/__init__.py:1090
msgid ""
"Installing on a USB device.  This may or may not produce a working system."
msgstr ""

#: pyanaconda/storage/__init__.py:1093
msgid ""
"Installing on a FireWire device.  This may or may not produce a working "
"system."
msgstr ""

#: pyanaconda/storage/__init__.py:1105
#, python-format
msgid ""
"You have not specified a swap partition.  %s MB of memory is required to "
"continue installation without a swap partition, but you only have %s MB."
msgstr ""

#: pyanaconda/storage/__init__.py:1110
msgid ""
"You have not specified a swap partition.  Although not strictly required in "
"all cases, it will significantly improve performance for most installations."
msgstr ""

#: pyanaconda/storage/__init__.py:1117
#, python-format
msgid ""
"This mount point is invalid.  The %s directory must be on the / file system."
msgstr ""

#: pyanaconda/storage/__init__.py:1121
#, python-format
msgid "The mount point %s must be on a linux file system."
msgstr ""

#: pyanaconda/storage/__init__.py:1132
msgid "No Drives Found"
msgstr ""

#: pyanaconda/storage/__init__.py:1133
msgid ""
"An error has occurred - no valid devices were found on which to create new "
"file systems. Please check your hardware for the cause of this problem."
msgstr ""

#: pyanaconda/storage/__init__.py:1421 pyanaconda/storage/__init__.py:1430
msgid "Dirty File Systems"
msgstr ""

#: pyanaconda/storage/__init__.py:1422
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Please boot your Linux installation, let the file systems be "
"checked and shut down cleanly to upgrade.\n"
"%s"
msgstr ""

#: pyanaconda/storage/__init__.py:1431
#, python-format
msgid ""
"The following file systems for your Linux system were not unmounted "
"cleanly.  Would you like to mount them anyway?\n"
"%s"
msgstr ""

#: pyanaconda/storage/__init__.py:1884
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"is an old-style Linux swap partition.  If you want to use this device for "
"swap space, you must reformat as a new-style Linux swap partition."
msgstr ""

#: pyanaconda/storage/__init__.py:1895
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. To perform an upgrade, please shut "
"down your system rather than hibernating it."
msgstr ""

#: pyanaconda/storage/__init__.py:1903
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"in your /etc/fstab file is currently in use as a software suspend device, "
"which means your system is hibernating. If you are performing a new install, "
"make sure the installer is set to format all swap devices."
msgstr ""

#: pyanaconda/storage/__init__.py:1915
#, python-format
msgid ""
"The swap device:\n"
"\n"
"     %s\n"
"\n"
"does not contain a supported swap volume.  In order to continue "
"installation, you will need to format the device or skip it."
msgstr ""

#: pyanaconda/storage/__init__.py:1926
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"The /etc/fstab on your upgrade partition does not reference a valid swap "
"device.\n"
"\n"
"Press OK to exit the installer"
msgstr ""

#: pyanaconda/storage/__init__.py:1933
#, python-format
msgid ""
"Error enabling swap device %(name)s: %(msg)s\n"
"\n"
"This most likely means this swap device has not been initialized.\n"
"\n"
"Press OK to exit the installer."
msgstr ""

#: pyanaconda/storage/__init__.py:1997 pyanaconda/storage/__init__.py:2009
msgid "Invalid mount point"
msgstr ""

#: pyanaconda/storage/__init__.py:1998
#, python-format
msgid ""
"An error occurred when trying to create %s.  Some element of this path is "
"not a directory. This is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: pyanaconda/storage/__init__.py:2010
#, python-format
msgid ""
"An error occurred when trying to create %(mountpoint)s: %(msg)s.  This is a "
"fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: pyanaconda/storage/__init__.py:2026 pyanaconda/storage/__init__.py:2050
msgid "Unable to mount filesystem"
msgstr ""

#: pyanaconda/storage/__init__.py:2027
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s.  You may "
"continue installation, but there may be problems."
msgstr ""

#: pyanaconda/storage/__init__.py:2051
#, python-format
msgid ""
"An error occurred mounting device %(path)s as %(mountpoint)s: %(msg)s. This "
"is a fatal error and the install cannot continue.\n"
"\n"
"Press <Enter> to exit the installer."
msgstr ""

#: pyanaconda/storage/dasd.py:155
msgid "Formatting DASD Device"
msgid_plural "Formatting DASD Devices"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/storage/dasd.py:156
#, python-format
msgid "Preparing %d DASD device for use with Linux..."
msgid_plural "Preparing %d DASD devices for use with Linux..."
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/storage/devicelibs/lvm.py:299
#, python-format
msgid "vginfo failed for %s"
msgstr ""

#: pyanaconda/storage/devicelibs/lvm.py:328
#, python-format
msgid "lvs failed for %s"
msgstr ""

#: pyanaconda/storage/devices.py:744
msgid "Creating"
msgstr ""

#: pyanaconda/storage/devices.py:745
#, python-format
msgid "Creating device %s"
msgstr ""

#: pyanaconda/storage/devices.py:2595
#, python-format
msgid "A RAID%d set requires at least %d member"
msgid_plural "A RAID%d set requires at least %d members"
msgstr[0] ""
msgstr[1] ""

#: pyanaconda/storage/devicetree.py:105
#, python-format
msgid ""
"Are you sure you want to skip entering a passphrase for device %s?\n"
"\n"
"If you skip this step the device's contents will not be available during "
"installation."
msgstr ""

#: pyanaconda/storage/devicetree.py:1640
msgid "This partition is part of an inconsistent LVM Volume Group."
msgstr ""

#: pyanaconda/storage/fcoe.py:66 pyanaconda/storage/fcoe.py:67
msgid "Connecting to FCoE SAN"
msgstr ""

#: pyanaconda/storage/fcoe.py:111
msgid "FCoE not available"
msgstr ""

#: pyanaconda/storage/formats/fs.py:102
msgid "filesystem configuration missing a type"
msgstr ""

#: pyanaconda/storage/formats/fs.py:354 pyanaconda/storage/formats/luks.py:194
#: pyanaconda/storage/formats/lvmpv.py:103
#: pyanaconda/storage/formats/swap.py:161
msgid "Formatting"
msgstr ""

#: pyanaconda/storage/formats/fs.py:355
#, python-format
msgid "Creating %(type)s filesystem on %(device)s"
msgstr ""

#: pyanaconda/storage/formats/fs.py:457
msgid "Resizing"
msgstr ""

#: pyanaconda/storage/formats/fs.py:458
#, python-format
msgid "Resizing filesystem on %s"
msgstr ""

#: pyanaconda/storage/formats/fs.py:493
#, python-format
msgid "Unknown return code: %d."
msgstr ""

#: pyanaconda/storage/formats/fs.py:507
msgid "Checking"
msgstr ""

#: pyanaconda/storage/formats/fs.py:508
#, python-format
msgid "Checking filesystem on %s"
msgstr ""

#: pyanaconda/storage/formats/fs.py:525
#, python-format
msgid "%(type)s filesystem check failure on %(device)s: "
msgstr ""

#: pyanaconda/storage/formats/fs.py:531
msgid ""
"Errors like this usually mean there is a problem with the filesystem that "
"will require user interaction to repair.  Before restarting installation, "
"reboot to rescue mode or another system that allows you to repair the "
"filesystem interactively.  Restart installation after you have corrected the "
"problems on the filesystem."
msgstr ""

#: pyanaconda/storage/formats/fs.py:539 pyanaconda/storage/partitioning.py:282
msgid "Unrecoverable Error"
msgstr ""

#: pyanaconda/storage/formats/fs.py:886
msgid "File system errors left uncorrected."
msgstr ""

#: pyanaconda/storage/formats/fs.py:887
msgid "Operational error."
msgstr ""

#: pyanaconda/storage/formats/fs.py:888
msgid "Usage or syntax error."
msgstr ""

#: pyanaconda/storage/formats/fs.py:889
msgid "e2fsck cancelled by user request."
msgstr ""

#: pyanaconda/storage/formats/fs.py:890
msgid "Shared library error."
msgstr ""

#: pyanaconda/storage/formats/fs.py:1076
msgid ""
"Recoverable errors have been detected or dosfsck has discovered an internal "
"inconsistency."
msgstr ""

#: pyanaconda/storage/formats/fs.py:1078
msgid "Usage error."
msgstr ""

#: pyanaconda/storage/formats/luks.py:48
msgid "Encrypted"
msgstr ""

#: pyanaconda/storage/formats/luks.py:195
#, python-format
msgid "Encrypting %s"
msgstr ""

#: pyanaconda/storage/formats/lvmpv.py:104
#: pyanaconda/storage/formats/swap.py:162
#, python-format
msgid "Creating %(type)s on %(device)s"
msgstr ""

#: pyanaconda/storage/iscsi.py:120
msgid "Unable to change iSCSI initiator name once set"
msgstr ""

#: pyanaconda/storage/iscsi.py:122
msgid "Must provide an iSCSI initiator name"
msgstr ""

#: pyanaconda/storage/iscsi.py:153 pyanaconda/storage/iscsi.py:154
msgid "Scanning iSCSI nodes"
msgstr ""

#: pyanaconda/storage/iscsi.py:174 pyanaconda/storage/iscsi.py:175
msgid "Initializing iSCSI initiator"
msgstr ""

#: pyanaconda/storage/iscsi.py:230
msgid "iSCSI not available"
msgstr ""

#: pyanaconda/storage/iscsi.py:232
msgid "No initiator name set"
msgstr ""

#: pyanaconda/storage/iscsi.py:258
msgid "Logging in to iSCSI node"
msgstr ""

#: pyanaconda/storage/iscsi.py:259
#, python-format
msgid "Logging in to iSCSI node %s"
msgstr ""

#: pyanaconda/storage/partitioning.py:200
msgid ""
"Could not find enough free space for automatic partitioning.  Press 'OK' to "
"exit the installer."
msgstr ""

#: pyanaconda/storage/partitioning.py:203
msgid ""
"Could not find enough free space for automatic partitioning, please use "
"another partitioning method."
msgstr ""

#: pyanaconda/storage/partitioning.py:231
msgid "Warnings During Automatic Partitioning"
msgstr ""

#: pyanaconda/storage/partitioning.py:233
#, python-format
msgid ""
"Following warnings occurred during automatic partitioning:\n"
"\n"
"%s"
msgstr ""

#: pyanaconda/storage/partitioning.py:247
#: pyanaconda/storage/partitioning.py:266
msgid ""
"\n"
"\n"
"Press 'OK' to exit the installer."
msgstr ""

#: pyanaconda/storage/partitioning.py:249
#, python-format
msgid ""
"Could not allocate requested partitions: \n"
"\n"
"%(msg)s.%(extra)s"
msgstr ""

#: pyanaconda/storage/partitioning.py:268
msgid ""
"\n"
"\n"
"Press 'OK' to choose a different partitioning option."
msgstr ""

#: pyanaconda/storage/partitioning.py:270
msgid "Automatic Partitioning Errors"
msgstr ""

#: pyanaconda/storage/partitioning.py:271
#, python-format
msgid ""
"The following errors occurred with your partitioning:\n"
"\n"
"%(errortxt)s\n"
"\n"
"This can happen if there is not enough space on your hard drive(s) for the "
"installation. %(extra)s"
msgstr ""

#: pyanaconda/storage/zfcp.py:51
msgid "You have not specified a device number or the number is invalid"
msgstr ""

#: pyanaconda/storage/zfcp.py:53
msgid "You have not specified a worldwide port name or the name is invalid."
msgstr ""

#: pyanaconda/storage/zfcp.py:55
msgid "You have not specified a FCP LUN or the number is invalid."
msgstr ""

#: pyanaconda/storage/zfcp.py:133
#, python-format
msgid "zFCP device %s not found, not even in device ignore list."
msgstr ""

#: pyanaconda/storage/zfcp.py:143
#, python-format
msgid "Could not set zFCP device %(devnum)s online (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:154
#, python-format
msgid "Could not add WWPN %(wwpn)s to zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:161
#, python-format
msgid "WWPN %(wwpn)s not found at zFCP device %(devnum)s."
msgstr ""

#: pyanaconda/storage/zfcp.py:176
#, python-format
msgid ""
"Could not add LUN %(fcplun)s to WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:182
#, python-format
msgid ""
"LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s already configured."
msgstr ""

#: pyanaconda/storage/zfcp.py:194
#, python-format
msgid ""
"Could not read failed attribute of LUN %(fcplun)s at WWPN %(wwpn)s on zFCP "
"device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:203
#, python-format
msgid ""
"Failed LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s removed "
"again."
msgstr ""

#: pyanaconda/storage/zfcp.py:260
#, python-format
msgid ""
"Could not correctly delete SCSI device of zFCP %(devnum)s %(wwpn)s "
"%(fcplun)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:269
#, python-format
msgid ""
"Could not remove LUN %(fcplun)s at WWPN %(wwpn)s on zFCP device %(devnum)s "
"(%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:287
#, python-format
msgid "Could not remove WWPN %(wwpn)s on zFCP device %(devnum)s (%(e)s)."
msgstr ""

#: pyanaconda/storage/zfcp.py:313
#, python-format
msgid "Could not set zFCP device %(devnum)s offline (%(e)s)."
msgstr ""

#: pyanaconda/textw/complete_text.py:32
msgid "Reboot"
msgstr ""

#: pyanaconda/textw/complete_text.py:34
msgid "<Enter> to exit"
msgstr ""

#: pyanaconda/textw/complete_text.py:62
msgid "Complete"
msgstr ""

#: pyanaconda/textw/constants_text.py:75
#, python-format
msgid ""
"This storage device may contain data:\n"
"\n"
"%s, %s MB, %s\n"
"\n"
"We could not detect partitions or filesystems on this device. This could be "
"because the device is blank, unpartitioned, or virtual. If not, there may be "
"data on the device that can not be recovered if you use it in this "
"installation. We can remove the device from this installation to protect the "
"data.\n"
"\n"
"Are you sure this device does not contain valuable data?"
msgstr ""

#: pyanaconda/textw/constants_text.py:88
msgid "Apply my choice to all devices with undetected partitions."
msgstr ""

#: pyanaconda/textw/keyboard_text.py:48
msgid "Keyboard Selection"
msgstr ""

#: pyanaconda/textw/keyboard_text.py:49
msgid "Which model keyboard is attached to this computer?"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:35 pyanaconda/textw/netconfig_text.py:42
msgid "Error With Data"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:36
#, python-format
msgid ""
"An error occurred converting the value entered for \"%(field)s\":\n"
"%(errmsg)s"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:43
#, python-format
msgid "A value is required for the field %s"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:126
#: pyanaconda/textw/netconfig_text.py:152
msgid "Enable network interface"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:129
msgid ""
"This requires that you have an active network connection during the "
"installation process.  Please select network interface to configure."
msgstr ""

#: pyanaconda/textw/netconfig_text.py:155
#, python-format
msgid "Configure interface %s to be used during installation process."
msgstr ""

#: pyanaconda/textw/netconfig_text.py:160
msgid "Enable IPv4 Support"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:167
#: pyanaconda/textw/netconfig_text.py:207
msgid "Manual Configuration"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:175
msgid "IPv4 Address:"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:189
#: pyanaconda/textw/netconfig_text.py:229
msgid "Nameserver:"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:197
msgid "Enable IPv6 Support"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:204
msgid "Dynamic IP Configuration (DHCPv6)"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:215
msgid "IPv6 Address:"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:270
msgid "Missing protocol"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:271
msgid "You must select at least one protocol version"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:279
#: pyanaconda/textw/netconfig_text.py:282
msgid "IPv4 Address"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:286
#: pyanaconda/textw/netconfig_text.py:291
msgid "IPv4 Prefix"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:290
msgid "IPv4 CIDR prefix must be between 0 and 32."
msgstr ""

#: pyanaconda/textw/netconfig_text.py:298
msgid "IPv4 Gateway"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:307
msgid "IPv4 Nameserver"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:315
#: pyanaconda/textw/netconfig_text.py:318
msgid "IPv6 Address"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:322
#: pyanaconda/textw/netconfig_text.py:327
msgid "IPv6 Prefix"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:326
msgid "IPv6 CIDR prefix must be between 0 and 128."
msgstr ""

#: pyanaconda/textw/netconfig_text.py:334
msgid "IPv6 Gateway"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:342
msgid "IPv6 Nameserver"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:399
msgid "Configuring Network Interfaces"
msgstr ""

#: pyanaconda/textw/netconfig_text.py:404
#, python-format
msgid "There was an error configuring network device %s"
msgstr ""

#: pyanaconda/textw/partition_text.py:59
msgid "Partitioning Type"
msgstr ""

#: pyanaconda/textw/partition_text.py:61
msgid ""
"Installation requires partitioning of your hard drive.  The default layout "
"is suitable for most users.  Select what space to use and which drives to "
"use as the install target."
msgstr ""

#: pyanaconda/textw/partition_text.py:64
msgid "Use entire drive"
msgstr ""

#: pyanaconda/textw/partition_text.py:65
msgid "Replace existing Linux system"
msgstr ""

#: pyanaconda/textw/partition_text.py:66
msgid "Use free space"
msgstr ""

#: pyanaconda/textw/partition_text.py:81
msgid "Which drive(s) do you want to use for this installation?"
msgstr ""

#: pyanaconda/textw/partition_text.py:96
msgid "<Space>,<+>,<-> selection   |   <F2> Add drive   |   <F12> next screen"
msgstr ""

#: pyanaconda/textw/progress_text.py:46
msgid "Package Installation"
msgstr ""

#: pyanaconda/textw/timezone_text.py:54
msgid "In which time zone are you located?"
msgstr ""

#: pyanaconda/textw/timezone_text.py:66
msgid "System clock uses UTC"
msgstr ""

#: pyanaconda/textw/upgrade_bootloader_text.py:111
#: pyanaconda/textw/upgrade_bootloader_text.py:121
msgid "Update boot loader configuration"
msgstr ""

#: pyanaconda/textw/upgrade_bootloader_text.py:124
msgid "Skip boot loader updating"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:125
msgid "Free Space"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:143
msgid "RAM detected (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:146
msgid "Suggested size (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:149
msgid "Swap file size (MB):"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:157
msgid "Add Swap"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:182
msgid "The value you entered is not a valid number."
msgstr ""

#: pyanaconda/textw/upgrade_text.py:214
msgid "Reinstall System"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:226
msgid "System to Upgrade"
msgstr ""

#: pyanaconda/textw/upgrade_text.py:227
msgid ""
"There seem to be one or more existing Linux installations on your system.\n"
"\n"
"Please choose one to upgrade, or select 'Reinstall System' to freshly "
"install your system."
msgstr ""

#: pyanaconda/textw/userauth_text.py:30
msgid "Root Password"
msgstr ""

#: pyanaconda/textw/userauth_text.py:33
msgid ""
"Pick a root password. You must type it twice to ensure you know it and do "
"not make a typing mistake. "
msgstr ""

#: pyanaconda/textw/userauth_text.py:67
msgid "The root password must be at least 6 characters long."
msgstr ""

#: pyanaconda/textw/userauth_text.py:84
#, python-format
msgid ""
"You have provided a weak password: %s\n"
"\n"
"Would you like to continue with this password?"
msgstr ""

#: pyanaconda/textw/zipl_text.py:36
msgid ""
"The z/IPL Boot Loader will be installed on your system after installation is "
"complete. You can now enter any additional kernel and chandev parameters "
"which your machine or your setup require."
msgstr ""

#: pyanaconda/textw/zipl_text.py:68
msgid "z/IPL Configuration"
msgstr ""

#: pyanaconda/textw/zipl_text.py:76 pyanaconda/textw/zipl_text.py:80
msgid "Chandev line "
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:1
msgid "Install"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:2
msgid "Install the live CD to your hard disk"
msgstr ""

#: data/liveinst/liveinst.desktop.in.h:3
msgid "Install to Hard Drive"
msgstr ""

#: liveinst:5
msgid "Starting Install to Hard Drive"
msgstr ""

#: data/ui/GroupSelector.glade.h:1
msgid ""
"Some packages associated with this group are not required to be installed "
"but may provide additional functionality.  Please choose the packages which "
"you would like to have installed."
msgstr ""

#: data/ui/GroupSelector.glade.h:2
msgid "_Deselect"
msgstr ""

#: data/ui/GroupSelector.glade.h:3
msgid "_Deselect all optional packages"
msgstr ""

#: data/ui/GroupSelector.glade.h:4
msgid "_Optional packages"
msgstr ""

#: data/ui/GroupSelector.glade.h:5
msgid "_Select"
msgstr ""

#: data/ui/GroupSelector.glade.h:6
msgid "_Select all optional packages"
msgstr ""

#: data/ui/GroupSelector.glade.h:7
msgid "dialog1"
msgstr ""

#: data/ui/account.glade.h:1
msgid "Confirm:"
msgstr ""

#: data/ui/account.glade.h:2
msgid "Root Password:"
msgstr ""

#: data/ui/account.glade.h:3
msgid ""
"The root account is used for administering the system.  Enter a password for "
"the root user."
msgstr ""

#: data/ui/adddrive.glade.h:1
msgid "Add _FCoE SAN"
msgstr ""

#: data/ui/adddrive.glade.h:2
msgid "Add _ZFCP LUN"
msgstr ""

#: data/ui/adddrive.glade.h:3
msgid "Add _iSCSI target"
msgstr ""

#: data/ui/adddrive.glade.h:4
msgid "Advanced Storage Options"
msgstr ""

#: data/ui/adddrive.glade.h:5
msgid "How would you like to modify your drive configuration?"
msgstr ""

#: data/ui/adddrive.glade.h:6
msgid "_Add drive"
msgstr ""

#: data/ui/addrepo.glade.h:1
msgid "<b>Repository _name:</b>"
msgstr ""

#: data/ui/addrepo.glade.h:2
msgid "<b>Repository _type:</b>"
msgstr ""

#: data/ui/addrepo.glade.h:4
msgid "Configure _proxy"
msgstr ""

#: data/ui/addrepo.glade.h:5
msgid ""
"HTTP/FTP\n"
"CD/DVD\n"
"NFS\n"
"Hard Drive"
msgstr ""

#: data/ui/addrepo.glade.h:9
msgid ""
"Please provide the configuration information for this software repository."
msgstr ""

#: data/ui/addrepo.glade.h:10
msgid "Proxy U_RL (host:port)"
msgstr ""

#: data/ui/addrepo.glade.h:11
msgid "Proxy pass_word"
msgstr ""

#: data/ui/addrepo.glade.h:12
msgid "Proxy u_sername"
msgstr ""

#: data/ui/addrepo.glade.h:13
msgid "Repository _URL"
msgstr ""

#: data/ui/addrepo.glade.h:14
msgid "Select A Directory"
msgstr ""

#: data/ui/addrepo.glade.h:15
msgid "URL is a _mirror list"
msgstr ""

#: data/ui/addrepo.glade.h:16
msgid "_Directory"
msgstr ""

#: data/ui/addrepo.glade.h:17
msgid "_Options"
msgstr ""

#: data/ui/addrepo.glade.h:18
msgid "_Partition"
msgstr ""

#: data/ui/addrepo.glade.h:19
msgid "_Path"
msgstr ""

#: data/ui/addrepo.glade.h:20
msgid "_Server"
msgstr ""

#: data/ui/anaconda.glade.h:1
msgid "Reboo_t"
msgstr ""

#: data/ui/anaconda.glade.h:4
msgid "_Next"
msgstr ""

#: data/ui/autopart.glade.h:1
msgid "<b>Shrink partition _to size (in MB):</b>"
msgstr ""

#: data/ui/autopart.glade.h:2
msgid "Re_view and modify partitioning layout"
msgstr ""

#: data/ui/autopart.glade.h:3
msgid "Volume to Shrink"
msgstr ""

#: data/ui/autopart.glade.h:4
msgid ""
"Which partition would you like to shrink to make room for your installation?"
msgstr ""

#: data/ui/autopart.glade.h:5
msgid "Which type of installation would you like?"
msgstr ""

#: data/ui/autopart.glade.h:6
msgid "_Encrypt system"
msgstr ""

#: data/ui/autopart.glade.h:7
msgid "_Shrink"
msgstr ""

#: data/ui/blwhere.glade.h:1
msgid "/boot"
msgstr ""

#: data/ui/blwhere.glade.h:2
msgid "BIOS Drive Order"
msgstr ""

#: data/ui/blwhere.glade.h:3
msgid "Boot loader device"
msgstr ""

#: data/ui/blwhere.glade.h:4
msgid "First BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:5
msgid "Fourth BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:6
msgid "MBR"
msgstr ""

#: data/ui/blwhere.glade.h:7
msgid "Second BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:8
msgid "Third BIOS drive:"
msgstr ""

#: data/ui/blwhere.glade.h:9
msgid "Where would you like to install the boot loader for your system?"
msgstr ""

#: data/ui/cleardisks.glade.h:1
msgid "<b>Data Storage Devices</b> (to be mounted only)"
msgstr ""

#: data/ui/cleardisks.glade.h:2
msgid "<b>Install Target Devices</b>"
msgstr ""

#: data/ui/cleardisks.glade.h:3
msgid ""
"<b>Tip:</b> Install target devices will be reformatted and wiped of any "
"data.  Make sure you have backups."
msgstr ""

#: data/ui/cleardisks.glade.h:4
msgid ""
"Below are the storage devices you've selected to be a part of this "
"installation.  Please indicate using the arrows below which devices you'd "
"like to use as data devices (these will not be formatted, only mounted) and "
"which devices you'd like to use as installation target devices (these may be "
"formatted).  Please also indicate which system drive will have the "
"bootloader installed."
msgstr ""

#: data/ui/create-storage.glade.h:1
msgid ""
"<span size=\"small\" color=\"gray\">Create a RAID formatted partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:2
msgid ""
"<span size=\"small\" color=\"gray\">Create a logical volume on selected "
"volume group</span>"
msgstr ""

#: data/ui/create-storage.glade.h:3
msgid ""
"<span size=\"small\" color=\"gray\">Create an LVM formatted partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:4
msgid ""
"<span size=\"small\" color=\"gray\">General purpose partition creation</span>"
msgstr ""

#: data/ui/create-storage.glade.h:5
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 1 free LVM formatted "
"partition</span>"
msgstr ""

#: data/ui/create-storage.glade.h:6
msgid ""
"<span size=\"small\" color=\"gray\">Requires at least 2 free RAID formatted "
"partitions</span>"
msgstr ""

#: data/ui/create-storage.glade.h:7
msgid "C_reate"
msgstr ""

#: data/ui/create-storage.glade.h:8
msgid "Create LVM"
msgstr ""

#: data/ui/create-storage.glade.h:9
msgid "Create Partition"
msgstr ""

#: data/ui/create-storage.glade.h:10
msgid "Create Software RAID"
msgstr ""

#: data/ui/create-storage.glade.h:11
msgid "Create Storage"
msgstr ""

#: data/ui/create-storage.glade.h:12
msgid "LVM Logical Volume"
msgstr ""

#: data/ui/create-storage.glade.h:13
msgid "LVM Physical Volume"
msgstr ""

#: data/ui/create-storage.glade.h:14
msgid "LVM Volume Group"
msgstr ""

#: data/ui/create-storage.glade.h:16
msgid "RAID Partition"
msgstr ""

#: data/ui/create-storage.glade.h:17
msgid "Standard Partition"
msgstr ""

#: data/ui/detailed-dialog.glade.h:1
msgid "Info"
msgstr ""

#: data/ui/detailed-dialog.glade.h:2
msgid "_Details"
msgstr ""

#: data/ui/fcoe-config.glade.h:1
msgid "Configure FCoE Parameters"
msgstr ""

#: data/ui/fcoe-config.glade.h:2
msgid "NIC:"
msgstr ""

#: data/ui/fcoe-config.glade.h:3
msgid ""
"Please select the network interface which is connected to\n"
"your FCoE switch."
msgstr ""

#: data/ui/fcoe-config.glade.h:5
msgid "Use DCB"
msgstr ""

#: data/ui/fcoe-config.glade.h:6
msgid "_Add FCoE Disk(s)"
msgstr ""

#: data/ui/filter.glade.h:1
msgid ""
"<b>Tip:</b> Selected drives will not be formatted unless you choose to do so "
"later during installation.  Also, note that post-installation you may mount "
"drives you did not select here by modifying your /etc/fstab file."
msgstr ""

#: data/ui/filter.glade.h:2
msgid "Basic Devices"
msgstr ""

#: data/ui/filter.glade.h:3
msgid "Filter By:"
msgstr ""

#: data/ui/filter.glade.h:4
msgid "Firmware RAID"
msgstr ""

#: data/ui/filter.glade.h:5
msgid "Identifier:"
msgstr ""

#: data/ui/filter.glade.h:6
msgid ""
"Interconnect\n"
"Vendor\n"
"Identifier"
msgstr ""

#: data/ui/filter.glade.h:9
msgid "LUN:"
msgstr ""

#: data/ui/filter.glade.h:10
msgid "Multipath Devices"
msgstr ""

#: data/ui/filter.glade.h:11
msgid "Other SAN Devices"
msgstr ""

#: data/ui/filter.glade.h:12
msgid ""
"Please select the drives you'd like to install the operating system on, as "
"well as any drives you'd like to automatically mount on your system:"
msgstr ""

#: data/ui/filter.glade.h:13
msgid ""
"Port / Target / LUN\n"
"Target Identifier"
msgstr ""

#: data/ui/filter.glade.h:15
msgid "Port:"
msgstr ""

#: data/ui/filter.glade.h:16
msgid "Search"
msgstr ""

#: data/ui/filter.glade.h:17
msgid "Search By:"
msgstr ""

#: data/ui/filter.glade.h:18
msgid "Search Results:"
msgstr ""

#: data/ui/filter.glade.h:20
#, no-c-format
msgid "Selected devices: %s (%s) out of %s (%s)."
msgstr ""

#: data/ui/filter.glade.h:21
msgid "Show Identifiers that Include:"
msgstr ""

#: data/ui/filter.glade.h:22
msgid "Show Only Devices From:"
msgstr ""

#: data/ui/filter.glade.h:23
msgid "Show Only Devices Using:"
msgstr ""

#: data/ui/filter.glade.h:24
msgid "Target:"
msgstr ""

#: data/ui/filter.glade.h:25
msgid "_Add Advanced Target"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:1
msgid "CHAP Password:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:2
msgid "CHAP Username:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:3
msgid "Check the nodes you wish to log into:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:4
msgid "Reason:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:5
msgid "Retry _Login"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:6
msgid "Reverse CHAP Password:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:7
msgid "Reverse CHAP Username:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:8
msgid "Start _Discovery"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:9
msgid "Successfully logged in and attached the following nodes:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:10
msgid "Target IP Address:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:11
msgid ""
"To use iSCSI disks, you must provide the address of your iSCSI target and "
"the iSCSI initiator name you've configured for your host."
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:12
msgid ""
"What kind of iSCSI <b>discovery authentication</b> do you wish to perform:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:13
msgid "What kind of iSCSI <b>login authentication</b> do you wish to perform:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:14
msgid "_Cancel"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:15
msgid "_Login"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:16
msgid "_OK"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:17
msgid "iSCSI Discovered Nodes"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:18
msgid "iSCSI Discovery Details"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:19
msgid "iSCSI Initiator Name:"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:20
msgid "iSCSI Login Results"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:21
msgid "iSCSI Nodes Login"
msgstr ""

#: data/ui/iscsi-dialogs.glade.h:22
msgid "iSCSI login has failed for the following nodes:"
msgstr ""

#: data/ui/lukspassphrase.glade.h:1
msgid ""
"Also add this passphrase to all existing encrypted devices to streamline the "
"boot process"
msgstr ""

#: data/ui/lukspassphrase.glade.h:2
msgid ""
"Choose a passphrase for this encrypted partition. You will be prompted for "
"the passphrase during system boot."
msgstr ""

#: data/ui/lukspassphrase.glade.h:3
msgid "Confirm passphrase:"
msgstr ""

#: data/ui/lukspassphrase.glade.h:4
msgid "Enter passphrase for encrypted partition"
msgstr ""

#: data/ui/lukspassphrase.glade.h:5
msgid "Enter passphrase:"
msgstr ""

#: data/ui/network.glade.h:1
msgid "Configure Network"
msgstr ""

#: data/ui/network.glade.h:2
msgid "Hostname:"
msgstr ""

#: data/ui/network.glade.h:3
msgid ""
"Please name this computer.  The hostname identifies the computer on a "
"network."
msgstr ""

#: data/ui/tasksel.glade.h:1
msgid "Customize _later"
msgstr ""

#: data/ui/tasksel.glade.h:2
msgid ""
"Please select any additional repositories that you want to use for software "
"installation."
msgstr ""

#: data/ui/tasksel.glade.h:4
#, no-c-format
msgid ""
"The default installation of %s includes a set of software applicable for "
"general internet usage.  What additional tasks would you like your system to "
"support?"
msgstr ""

#: data/ui/tasksel.glade.h:5
msgid ""
"You can further customize the software selection now, or after install via "
"the software management application."
msgstr ""

#: data/ui/tasksel.glade.h:6
msgid "_Add additional software repositories"
msgstr ""

#: data/ui/tasksel.glade.h:7
msgid "_Customize now"
msgstr ""

#: data/ui/tasksel.glade.h:8
msgid "_Modify repository"
msgstr ""

#: data/ui/zfcp-config.glade.h:1
msgid "<b>Device number:</b>"
msgstr ""

#: data/ui/zfcp-config.glade.h:2
msgid "<b>FCP LUN:</b>"
msgstr ""

#: data/ui/zfcp-config.glade.h:3
msgid "<b>WWPN:</b>"
msgstr ""

#: data/ui/zfcp-config.glade.h:4
msgid "Add FCP device"
msgstr ""

#: data/ui/zfcp-config.glade.h:5
msgid ""
"zSeries machines can access industry-standard SCSI devices via Fibre Channel "
"(FCP). You need to provide a 16 bit device number, a 64 bit World Wide Port "
"Name (WWPN), and a 64 bit FCP LUN for each device."
msgstr ""

#: data/ui/reinitialize-dialog.glade.h:1
msgid ""
"<span size=\"large\" weight=\"bold\">The storage device below may contain "
"data.</span>"
msgstr ""

#: data/ui/reinitialize-dialog.glade.h:2
msgid ""
"Apply my choice to all devices with undetected partitions or filesystems."
msgstr ""

#: data/ui/reinitialize-dialog.glade.h:4
msgid ""
"We could not detect partitions or filesystems on this device.\n"
"\n"
"This could be because the device is <b>blank</b>, <b>unpartitioned</b>, or "
"<b>virtual</b>. If not,\n"
"there may be data on the device that can not be recovered if you use it in "
"this\n"
"installation. We can remove the device from this installation to protect the "
"data.\n"
"\n"
"Are you sure this device does not contain valuable data?"
msgstr ""

#: data/ui/reinitialize-dialog.glade.h:11
msgid "_No, keep any data"
msgstr ""

#: data/ui/reinitialize-dialog.glade.h:12
msgid "_Yes, discard any data"
msgstr ""

#: data/lang-table.h:1
msgid "Afrikaans"
msgstr ""

#: data/lang-table.h:2
msgid "Arabic"
msgstr ""

#: data/lang-table.h:3
msgid "Assamese"
msgstr ""

#: data/lang-table.h:4
msgid "Bengali"
msgstr ""

#: data/lang-table.h:5
msgid "Bengali(India)"
msgstr ""

#: data/lang-table.h:6
msgid "Bulgarian"
msgstr ""

#: data/lang-table.h:7
msgid "Catalan"
msgstr ""

#: data/lang-table.h:8
msgid "Chinese(Simplified)"
msgstr ""

#: data/lang-table.h:9
msgid "Chinese(Traditional)"
msgstr ""

#: data/lang-table.h:10
msgid "Croatian"
msgstr ""

#: data/lang-table.h:11
msgid "Czech"
msgstr ""

#: data/lang-table.h:12
msgid "Danish"
msgstr ""

#: data/lang-table.h:13
msgid "Dutch"
msgstr ""

#: data/lang-table.h:14
msgid "English"
msgstr ""

#: data/lang-table.h:15
msgid "Estonian"
msgstr ""

#: data/lang-table.h:16
msgid "Finnish"
msgstr ""

#: data/lang-table.h:17
msgid "French"
msgstr ""

#: data/lang-table.h:18
msgid "German"
msgstr ""

#: data/lang-table.h:19
msgid "Greek"
msgstr ""

#: data/lang-table.h:20
msgid "Gujarati"
msgstr ""

#: data/lang-table.h:21
msgid "Hebrew"
msgstr ""

#: data/lang-table.h:22
msgid "Hindi"
msgstr ""

#: data/lang-table.h:23
msgid "Hungarian"
msgstr ""

#: data/lang-table.h:24
msgid "Icelandic"
msgstr ""

#: data/lang-table.h:25
msgid "Iloko"
msgstr ""

#: data/lang-table.h:26
msgid "Indonesian"
msgstr ""

#: data/lang-table.h:27
msgid "Italian"
msgstr ""

#: data/lang-table.h:28
msgid "Japanese"
msgstr ""

#: data/lang-table.h:29
msgid "Kannada"
msgstr ""

#: data/lang-table.h:30
msgid "Korean"
msgstr ""

#: data/lang-table.h:31
msgid "Macedonian"
msgstr ""

#: data/lang-table.h:32
msgid "Maithili"
msgstr ""

#: data/lang-table.h:33
msgid "Malay"
msgstr ""

#: data/lang-table.h:34
msgid "Malayalam"
msgstr ""

#: data/lang-table.h:35
msgid "Marathi"
msgstr ""

#: data/lang-table.h:36
msgid "Nepali"
msgstr ""

#: data/lang-table.h:37
msgid "Norwegian(Bokmål)"
msgstr ""

#: data/lang-table.h:38
msgid "Northern Sotho"
msgstr ""

#: data/lang-table.h:39
msgid "Oriya"
msgstr ""

#: data/lang-table.h:40
msgid "Persian"
msgstr ""

#: data/lang-table.h:41
msgid "Polish"
msgstr ""

#: data/lang-table.h:42
msgid "Portuguese"
msgstr ""

#: data/lang-table.h:43
msgid "Portuguese(Brazilian)"
msgstr ""

#: data/lang-table.h:44
msgid "Punjabi"
msgstr ""

#: data/lang-table.h:45
msgid "Romanian"
msgstr ""

#: data/lang-table.h:46
msgid "Russian"
msgstr ""

#: data/lang-table.h:47
msgid "Serbian"
msgstr ""

#: data/lang-table.h:48
msgid "Serbian(Latin)"
msgstr ""

#: data/lang-table.h:49
msgid "Sinhala"
msgstr ""

#: data/lang-table.h:50
msgid "Slovak"
msgstr ""

#: data/lang-table.h:51
msgid "Slovenian"
msgstr ""

#: data/lang-table.h:52
msgid "Spanish"
msgstr ""

#: data/lang-table.h:53
msgid "Swedish"
msgstr ""

#: data/lang-table.h:54
msgid "Tajik"
msgstr ""

#: data/lang-table.h:55
msgid "Tamil"
msgstr ""

#: data/lang-table.h:56
msgid "Telugu"
msgstr ""

#: data/lang-table.h:57
msgid "Turkish"
msgstr ""

#: data/lang-table.h:58
msgid "Ukrainian"
msgstr ""

#: data/lang-table.h:59
msgid "Vietnamese"
msgstr ""

#: data/lang-table.h:60
msgid "Welsh"
msgstr ""

#: data/lang-table.h:61
msgid "Zulu"
msgstr ""
